roachtest: liquibase failed

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5500557?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5500557?buildTab=artifacts#/liquibase) on master @ [8d34ef1ea15850ee1c70470610b6652df4c317de](https://github.com/cockroachdb/cockroach/commits/8d34ef1ea15850ee1c70470610b6652df4c317de):


```
test artifacts and logs in: /artifacts/liquibase/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,liquibase.go:123,liquibase.go:140,test_runner.go:897: 
		Tests run on Cockroach v22.2.0-alpha.00000000-811-g8d34ef1ea1
		Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
		47 Total Tests Run
		38 tests passed
		9 tests failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
		For a full summary look at the liquibase artifacts 
		
		An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1 release-blocker]
</p>
</details>
/cc @cockroachdb/sql-experience
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>


Jira issue: CRDB-16821

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5505694?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5505694?buildTab=artifacts#/liquibase) on master @ [8d34ef1ea15850ee1c70470610b6652df4c317de](https://github.com/cockroachdb/cockroach/commits/8d34ef1ea15850ee1c70470610b6652df4c317de):


```
test artifacts and logs in: /artifacts/liquibase/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,liquibase.go:123,liquibase.go:140,test_runner.go:897: 
		Tests run on Cockroach v22.2.0-alpha.00000000-811-g8d34ef1ea1
		Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
		47 Total Tests Run
		38 tests passed
		9 tests failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
		For a full summary look at the liquibase artifacts 
		
		An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1 release-blocker]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5511011?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5511011?buildTab=artifacts#/liquibase) on master @ [8d34ef1ea15850ee1c70470610b6652df4c317de](https://github.com/cockroachdb/cockroach/commits/8d34ef1ea15850ee1c70470610b6652df4c317de):


```
test artifacts and logs in: /artifacts/liquibase/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,liquibase.go:123,liquibase.go:140,test_runner.go:897: 
		Tests run on Cockroach v22.2.0-alpha.00000000-811-g8d34ef1ea1
		Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
		47 Total Tests Run
		38 tests passed
		9 tests failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
		For a full summary look at the liquibase artifacts 
		
		An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1 release-blocker]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5527687?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5527687?buildTab=artifacts#/liquibase) on master @ [13cb2f6c40e3146fed8d931f65f89da9b42ce2c1](https://github.com/cockroachdb/cockroach/commits/13cb2f6c40e3146fed8d931f65f89da9b42ce2c1):


```
test artifacts and logs in: /artifacts/liquibase/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,liquibase.go:123,liquibase.go:140,test_runner.go:896: 
		Tests run on Cockroach v22.2.0-alpha.00000000-834-g13cb2f6c40
		Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
		47 Total Tests Run
		38 tests passed
		9 tests failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
		For a full summary look at the liquibase artifacts 
		
		An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5538979?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5538979?buildTab=artifacts#/liquibase) on master @ [7a1ced4a49d28f66f381e78733bcab510e2cbafe](https://github.com/cockroachdb/cockroach/commits/7a1ced4a49d28f66f381e78733bcab510e2cbafe):


```
test artifacts and logs in: /artifacts/liquibase/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,liquibase.go:123,liquibase.go:140,test_runner.go:896: 
		Tests run on Cockroach v22.2.0-alpha.00000000-895-g7a1ced4a49
		Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
		47 Total Tests Run
		38 tests passed
		9 tests failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
		For a full summary look at the liquibase artifacts 
		
		An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5549092?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5549092?buildTab=artifacts#/liquibase) on master @ [7af6c0d3e2085cafda90d5f0b15d06f6a360f154](https://github.com/cockroachdb/cockroach/commits/7af6c0d3e2085cafda90d5f0b15d06f6a360f154):


```
test artifacts and logs in: /artifacts/liquibase/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,liquibase.go:123,liquibase.go:140,test_runner.go:896: 
		Tests run on Cockroach v22.2.0-alpha.00000000-935-g7af6c0d3e2
		Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
		47 Total Tests Run
		38 tests passed
		9 tests failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
		For a full summary look at the liquibase artifacts 
		
		An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5558662?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5558662?buildTab=artifacts#/liquibase) on master @ [fc7ae339c3f85ad2a2b28643e47d7c20768fe237](https://github.com/cockroachdb/cockroach/commits/fc7ae339c3f85ad2a2b28643e47d7c20768fe237):


```
test artifacts and logs in: /artifacts/liquibase/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,liquibase.go:123,liquibase.go:140,test_runner.go:896: 
		Tests run on Cockroach v22.2.0-alpha.00000000-968-gfc7ae339c3
		Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
		47 Total Tests Run
		38 tests passed
		9 tests failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
		For a full summary look at the liquibase artifacts 
		
		An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5564213?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5564213?buildTab=artifacts#/liquibase) on master @ [b47dc374417fe4ba83570784040da5650238b30e](https://github.com/cockroachdb/cockroach/commits/b47dc374417fe4ba83570784040da5650238b30e):


```
test artifacts and logs in: /artifacts/liquibase/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,liquibase.go:123,liquibase.go:140,test_runner.go:896: 
		Tests run on Cockroach v22.2.0-alpha.00000000-978-gb47dc37441
		Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
		47 Total Tests Run
		38 tests passed
		9 tests failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
		For a full summary look at the liquibase artifacts 
		
		An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5570107?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5570107?buildTab=artifacts#/liquibase) on master @ [a611410e3babe97a14152381b89060e1bafa2e27](https://github.com/cockroachdb/cockroach/commits/a611410e3babe97a14152381b89060e1bafa2e27):


```
test artifacts and logs in: /artifacts/liquibase/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,liquibase.go:123,liquibase.go:140,test_runner.go:896: 
		Tests run on Cockroach v22.2.0-alpha.00000000-984-ga611410e3b
		Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
		47 Total Tests Run
		38 tests passed
		9 tests failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
		For a full summary look at the liquibase artifacts 
		
		An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5582417?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5582417?buildTab=artifacts#/liquibase) on master @ [7f8fef4c3f24ec2b6f9a3db1eb38756da68b3d5a](https://github.com/cockroachdb/cockroach/commits/7f8fef4c3f24ec2b6f9a3db1eb38756da68b3d5a):


```
test artifacts and logs in: /artifacts/liquibase/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,liquibase.go:123,liquibase.go:140,test_runner.go:896: 
		Tests run on Cockroach v22.2.0-alpha.00000000-1010-g7f8fef4c3f
		Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
		47 Total Tests Run
		38 tests passed
		9 tests failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
		For a full summary look at the liquibase artifacts 
		
		An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5594177?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5594177?buildTab=artifacts#/liquibase) on master @ [ac11a55499120ffd33016feebc47b4ed43a21007](https://github.com/cockroachdb/cockroach/commits/ac11a55499120ffd33016feebc47b4ed43a21007):


```
test artifacts and logs in: /artifacts/liquibase/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,liquibase.go:123,liquibase.go:140,test_runner.go:896: 
		Tests run on Cockroach v22.2.0-alpha.00000000-1052-gac11a55499
		Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
		47 Total Tests Run
		38 tests passed
		9 tests failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
		For a full summary look at the liquibase artifacts 
		
		An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5617815?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5617815?buildTab=artifacts#/liquibase) on master @ [78caa5c3a8ecd0d0a612798a106b626edb3db2ee](https://github.com/cockroachdb/cockroach/commits/78caa5c3a8ecd0d0a612798a106b626edb3db2ee):


```
test artifacts and logs in: /artifacts/liquibase/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,liquibase.go:123,liquibase.go:140,test_runner.go:896: 
		Tests run on Cockroach v22.2.0-alpha.00000000-1149-g78caa5c3a8
		Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
		47 Total Tests Run
		38 tests passed
		9 tests failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
		For a full summary look at the liquibase artifacts 
		
		An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5628111?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5628111?buildTab=artifacts#/liquibase) on master @ [4e8134033890c956d39020f28a0e99b21b32649f](https://github.com/cockroachdb/cockroach/commits/4e8134033890c956d39020f28a0e99b21b32649f):


```
test artifacts and logs in: /artifacts/liquibase/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,liquibase.go:123,liquibase.go:140,test_runner.go:896: 
		Tests run on Cockroach v22.2.0-alpha.00000000-1181-g4e81340338
		Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
		47 Total Tests Run
		38 tests passed
		9 tests failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
		For a full summary look at the liquibase artifacts 
		
		An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5633774?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5633774?buildTab=artifacts#/liquibase) on master @ [3c9b17113488d2ee6929936aa6ec48396f3ed71c](https://github.com/cockroachdb/cockroach/commits/3c9b17113488d2ee6929936aa6ec48396f3ed71c):


```
test artifacts and logs in: /artifacts/liquibase/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,liquibase.go:123,liquibase.go:140,test_runner.go:896: 
		Tests run on Cockroach v22.2.0-alpha.00000000-1185-g3c9b171134
		Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
		47 Total Tests Run
		38 tests passed
		9 tests failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
		For a full summary look at the liquibase artifacts 
		
		An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5639494?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5639494?buildTab=artifacts#/liquibase) on master @ [b9a165c03643ed43a0d45b16c0a795244543a2fe](https://github.com/cockroachdb/cockroach/commits/b9a165c03643ed43a0d45b16c0a795244543a2fe):


```
test artifacts and logs in: /artifacts/liquibase/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,liquibase.go:123,liquibase.go:140,test_runner.go:896: 
		Tests run on Cockroach v22.2.0-alpha.00000000-1188-gb9a165c036
		Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
		47 Total Tests Run
		38 tests passed
		9 tests failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
		For a full summary look at the liquibase artifacts 
		
		An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5645962?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5645962?buildTab=artifacts#/liquibase) on master @ [7a8cac9b11036dcf401453eb36e531fe381b70b4](https://github.com/cockroachdb/cockroach/commits/7a8cac9b11036dcf401453eb36e531fe381b70b4):


```
test artifacts and logs in: /artifacts/liquibase/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,liquibase.go:123,liquibase.go:140,test_runner.go:896: 
		Tests run on Cockroach v22.2.0-alpha.00000000-1195-g7a8cac9b11
		Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
		47 Total Tests Run
		38 tests passed
		9 tests failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
		For a full summary look at the liquibase artifacts 
		
		An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5658707?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5658707?buildTab=artifacts#/liquibase) on master @ [33d70998719051ee058bc9e516afa238ea7b7451](https://github.com/cockroachdb/cockroach/commits/33d70998719051ee058bc9e516afa238ea7b7451):


```
test artifacts and logs in: /artifacts/liquibase/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,liquibase.go:123,liquibase.go:140,test_runner.go:896: 
		Tests run on Cockroach v22.2.0-alpha.00000000-1232-g33d7099871
		Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
		47 Total Tests Run
		38 tests passed
		9 tests failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
		For a full summary look at the liquibase artifacts 
		
		An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5672143?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5672143?buildTab=artifacts#/liquibase) on master @ [03d5260a467b8640de41ee08054c1ce05f91966c](https://github.com/cockroachdb/cockroach/commits/03d5260a467b8640de41ee08054c1ce05f91966c):


```
test artifacts and logs in: /artifacts/liquibase/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,liquibase.go:123,liquibase.go:140,test_runner.go:896: 
		Tests run on Cockroach v22.2.0-alpha.00000000-1281-g03d5260a46
		Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
		47 Total Tests Run
		38 tests passed
		9 tests failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
		For a full summary look at the liquibase artifacts 
		
		An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5685337?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5685337?buildTab=artifacts#/liquibase) on master @ [88789feea153b61b379c0240a07482bdfaf0c932](https://github.com/cockroachdb/cockroach/commits/88789feea153b61b379c0240a07482bdfaf0c932):


```
test artifacts and logs in: /artifacts/liquibase/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,liquibase.go:123,liquibase.go:140,test_runner.go:896: 
		Tests run on Cockroach v22.2.0-alpha.00000000-1334-g88789feea1
		Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
		47 Total Tests Run
		38 tests passed
		9 tests failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
		For a full summary look at the liquibase artifacts 
		
		An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5695598?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5695598?buildTab=artifacts#/liquibase) on master @ [54b4fcaa231abd16260fb09722ed291dbd6a4900](https://github.com/cockroachdb/cockroach/commits/54b4fcaa231abd16260fb09722ed291dbd6a4900):


```
test artifacts and logs in: /artifacts/liquibase/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,liquibase.go:123,liquibase.go:140,test_runner.go:896: 
		Tests run on Cockroach v22.2.0-alpha.00000000-1365-g54b4fcaa23
		Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
		47 Total Tests Run
		38 tests passed
		9 tests failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
		For a full summary look at the liquibase artifacts 
		
		An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5701221?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5701221?buildTab=artifacts#/liquibase) on master @ [6374bd83f192bd509e50313acb431690b9c3f8ae](https://github.com/cockroachdb/cockroach/commits/6374bd83f192bd509e50313acb431690b9c3f8ae):


```
test artifacts and logs in: /artifacts/liquibase/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,liquibase.go:123,liquibase.go:140,test_runner.go:896: 
		Tests run on Cockroach v22.2.0-alpha.00000000-1369-g6374bd83f1
		Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
		47 Total Tests Run
		38 tests passed
		9 tests failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
		For a full summary look at the liquibase artifacts 
		
		An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5718837?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5718837?buildTab=artifacts#/liquibase) on master @ [571bfa3afb3858ae84d8a8fcdbb0a38e058402a5](https://github.com/cockroachdb/cockroach/commits/571bfa3afb3858ae84d8a8fcdbb0a38e058402a5):


```
test artifacts and logs in: /artifacts/liquibase/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,liquibase.go:123,liquibase.go:140,test_runner.go:896: 
		Tests run on Cockroach v22.2.0-alpha.00000000-1456-g571bfa3afb
		Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
		47 Total Tests Run
		38 tests passed
		9 tests failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
		For a full summary look at the liquibase artifacts 
		
		An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5731065?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5731065?buildTab=artifacts#/liquibase) on master @ [687171ac6c2cd9992486bb3b8c9d252ac95ca1cd](https://github.com/cockroachdb/cockroach/commits/687171ac6c2cd9992486bb3b8c9d252ac95ca1cd):


```
test artifacts and logs in: /artifacts/liquibase/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,liquibase.go:123,liquibase.go:140,test_runner.go:896: 
		Tests run on Cockroach v22.2.0-alpha.00000000-1498-g687171ac6c
		Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
		47 Total Tests Run
		38 tests passed
		9 tests failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
		For a full summary look at the liquibase artifacts 
		
		An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5741985?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5741985?buildTab=artifacts#/liquibase) on master @ [88d3253301457ac57820e0f4a4fab8f74bf9f38b](https://github.com/cockroachdb/cockroach/commits/88d3253301457ac57820e0f4a4fab8f74bf9f38b):


```
test artifacts and logs in: /artifacts/liquibase/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,liquibase.go:123,liquibase.go:140,test_runner.go:896: 
		Tests run on Cockroach v22.2.0-alpha.00000000-1523-g88d3253301
		Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
		47 Total Tests Run
		38 tests passed
		9 tests failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
		For a full summary look at the liquibase artifacts 
		
		An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5753958?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5753958?buildTab=artifacts#/liquibase) on master @ [e9ee21860458d997a8155734dc608cfcd050ef24](https://github.com/cockroachdb/cockroach/commits/e9ee21860458d997a8155734dc608cfcd050ef24):


```
test artifacts and logs in: /artifacts/liquibase/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,liquibase.go:123,liquibase.go:140,test_runner.go:896: 
		Tests run on Cockroach v22.2.0-alpha.00000000-1571-ge9ee218604
		Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
		47 Total Tests Run
		38 tests passed
		9 tests failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
		For a full summary look at the liquibase artifacts 
		
		An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5765317?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5765317?buildTab=artifacts#/liquibase) on master @ [0e816efe5443764591a944edcb385c7b2762523e](https://github.com/cockroachdb/cockroach/commits/0e816efe5443764591a944edcb385c7b2762523e):


```
test artifacts and logs in: /artifacts/liquibase/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,liquibase.go:123,liquibase.go:140,test_runner.go:896: 
		Tests run on Cockroach v22.2.0-alpha.00000000-1605-g0e816efe54
		Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
		47 Total Tests Run
		38 tests passed
		9 tests failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
		For a full summary look at the liquibase artifacts 
		
		An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5771178?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5771178?buildTab=artifacts#/liquibase) on master @ [e4cafeb8b1d586d091fb98e3e570650d7eeea294](https://github.com/cockroachdb/cockroach/commits/e4cafeb8b1d586d091fb98e3e570650d7eeea294):


```
test artifacts and logs in: /artifacts/liquibase/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,liquibase.go:123,liquibase.go:140,test_runner.go:896: 
		Tests run on Cockroach v22.2.0-alpha.00000000-1611-ge4cafeb8b1
		Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
		47 Total Tests Run
		38 tests passed
		9 tests failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
		For a full summary look at the liquibase artifacts 
		
		An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5776600?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5776600?buildTab=artifacts#/liquibase) on master @ [dc382b1ec54b4599cce1f5f50f243723a4286da6](https://github.com/cockroachdb/cockroach/commits/dc382b1ec54b4599cce1f5f50f243723a4286da6):


```
test artifacts and logs in: /artifacts/liquibase/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,liquibase.go:123,liquibase.go:140,test_runner.go:896: 
		Tests run on Cockroach v22.2.0-alpha.00000000-1613-gdc382b1ec5
		Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
		47 Total Tests Run
		38 tests passed
		9 tests failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
		For a full summary look at the liquibase artifacts 
		
		An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5789225?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5789225?buildTab=artifacts#/liquibase) on master @ [3dcf715f37595a73e8504b5331493f75207c6524](https://github.com/cockroachdb/cockroach/commits/3dcf715f37595a73e8504b5331493f75207c6524):


```
test artifacts and logs in: /artifacts/liquibase/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,liquibase.go:123,liquibase.go:140,test_runner.go:896: 
		Tests run on Cockroach v22.2.0-alpha.00000000-1680-g3dcf715f37
		Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
		47 Total Tests Run
		38 tests passed
		9 tests failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
		For a full summary look at the liquibase artifacts 
		
		An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5813917?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5813917?buildTab=artifacts#/liquibase) on master @ [457d724622e4fa2e62d6f4e7926509dbc7d18511](https://github.com/cockroachdb/cockroach/commits/457d724622e4fa2e62d6f4e7926509dbc7d18511):


```
test artifacts and logs in: /artifacts/liquibase/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,liquibase.go:123,liquibase.go:140,test_runner.go:896: 
		Tests run on Cockroach v22.2.0-alpha.00000000-1759-g457d724622
		Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
		47 Total Tests Run
		38 tests passed
		9 tests failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
		For a full summary look at the liquibase artifacts 
		
		An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5828568?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5828568?buildTab=artifacts#/liquibase) on master @ [fafc3330130a016a881592d0337ec48d9ce43b61](https://github.com/cockroachdb/cockroach/commits/fafc3330130a016a881592d0337ec48d9ce43b61):


```
test artifacts and logs in: /artifacts/liquibase/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,liquibase.go:123,liquibase.go:140,test_runner.go:896: 
		Tests run on Cockroach v22.2.0-alpha.00000000-1800-gfafc333013
		Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
		47 Total Tests Run
		38 tests passed
		9 tests failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
		For a full summary look at the liquibase artifacts 
		
		An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5840461?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5840461?buildTab=artifacts#/liquibase) on master @ [8fd5b3500796fae41c07fffd4246648b349b6460](https://github.com/cockroachdb/cockroach/commits/8fd5b3500796fae41c07fffd4246648b349b6460):


```
test artifacts and logs in: /artifacts/liquibase/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,liquibase.go:123,liquibase.go:140,test_runner.go:896: 
		Tests run on Cockroach v22.2.0-alpha.00000000-1833-g8fd5b35007
		Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
		47 Total Tests Run
		38 tests passed
		9 tests failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
		For a full summary look at the liquibase artifacts 
		
		An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5846431?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5846431?buildTab=artifacts#/liquibase) on master @ [eabdc49383a19d5731b765dbb0d8b45bd9e24404](https://github.com/cockroachdb/cockroach/commits/eabdc49383a19d5731b765dbb0d8b45bd9e24404):


```
test artifacts and logs in: /artifacts/liquibase/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,liquibase.go:123,liquibase.go:140,test_runner.go:896: 
		Tests run on Cockroach v22.2.0-alpha.00000000-1838-geabdc49383
		Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
		47 Total Tests Run
		38 tests passed
		9 tests failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
		For a full summary look at the liquibase artifacts 
		
		An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5851979?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5851979?buildTab=artifacts#/liquibase) on master @ [eabdc49383a19d5731b765dbb0d8b45bd9e24404](https://github.com/cockroachdb/cockroach/commits/eabdc49383a19d5731b765dbb0d8b45bd9e24404):


```
test artifacts and logs in: /artifacts/liquibase/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,liquibase.go:123,liquibase.go:140,test_runner.go:896: 
		Tests run on Cockroach v22.2.0-alpha.00000000-1838-geabdc49383
		Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
		47 Total Tests Run
		38 tests passed
		9 tests failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
		For a full summary look at the liquibase artifacts 
		
		An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5864984?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5864984?buildTab=artifacts#/liquibase) on master @ [8db26695114981eec5c12317f3dc5d770b522a3f](https://github.com/cockroachdb/cockroach/commits/8db26695114981eec5c12317f3dc5d770b522a3f):


```
test artifacts and logs in: /artifacts/liquibase/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,liquibase.go:123,liquibase.go:140,test_runner.go:896: 
		Tests run on Cockroach v22.2.0-alpha.00000000-1871-g8db2669511
		Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
		47 Total Tests Run
		38 tests passed
		9 tests failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
		For a full summary look at the liquibase artifacts 
		
		An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5875958?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5875958?buildTab=artifacts#/liquibase) on master @ [0f100f3a09246bc80ef2c3119e4ec1c24ef36801](https://github.com/cockroachdb/cockroach/commits/0f100f3a09246bc80ef2c3119e4ec1c24ef36801):


```
test artifacts and logs in: /artifacts/liquibase/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,liquibase.go:123,liquibase.go:140,test_runner.go:896: 
		Tests run on Cockroach v22.2.0-alpha.00000000-1898-g0f100f3a09
		Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
		47 Total Tests Run
		38 tests passed
		9 tests failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
		For a full summary look at the liquibase artifacts 
		
		An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5889604?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5889604?buildTab=artifacts#/liquibase) on master @ [7c95ccd0a9f0b0aca9cdb0ed813caf8d60b28732](https://github.com/cockroachdb/cockroach/commits/7c95ccd0a9f0b0aca9cdb0ed813caf8d60b28732):


```
test artifacts and logs in: /artifacts/liquibase/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,liquibase.go:123,liquibase.go:140,test_runner.go:896: 
		Tests run on Cockroach v22.2.0-alpha.00000000-1939-g7c95ccd0a9
		Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
		47 Total Tests Run
		38 tests passed
		9 tests failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
		For a full summary look at the liquibase artifacts 
		
		An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5902767?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5902767?buildTab=artifacts#/liquibase) on master @ [1129fbc650fe3a037b03aea1e5f1d8078618cb1c](https://github.com/cockroachdb/cockroach/commits/1129fbc650fe3a037b03aea1e5f1d8078618cb1c):


```
test artifacts and logs in: /artifacts/liquibase/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,liquibase.go:123,liquibase.go:140,test_runner.go:896: 
		Tests run on Cockroach v22.2.0-alpha.00000000-1970-g1129fbc650
		Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
		47 Total Tests Run
		38 tests passed
		9 tests failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
		For a full summary look at the liquibase artifacts 
		
		An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5915393?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5915393?buildTab=artifacts#/liquibase) on master @ [cd113bf428e8868c38b2f2f295210f5c30d13a2d](https://github.com/cockroachdb/cockroach/commits/cd113bf428e8868c38b2f2f295210f5c30d13a2d):


```
test artifacts and logs in: /artifacts/liquibase/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,liquibase.go:123,liquibase.go:140,test_runner.go:896: 
		Tests run on Cockroach v22.2.0-alpha.00000000-2040-gcd113bf428
		Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
		47 Total Tests Run
		38 tests passed
		9 tests failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
		For a full summary look at the liquibase artifacts 
		
		An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5921532?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5921532?buildTab=artifacts#/liquibase) on master @ [0352fbee5e6933e8bca28aae4b136bcd495ab227](https://github.com/cockroachdb/cockroach/commits/0352fbee5e6933e8bca28aae4b136bcd495ab227):


```
test artifacts and logs in: /artifacts/liquibase/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,liquibase.go:123,liquibase.go:140,test_runner.go:896: 
		Tests run on Cockroach v22.2.0-alpha.00000000-2042-g0352fbee5e
		Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
		47 Total Tests Run
		38 tests passed
		9 tests failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
		For a full summary look at the liquibase artifacts 
		
		An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5926514?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5926514?buildTab=artifacts#/liquibase) on master @ [0352fbee5e6933e8bca28aae4b136bcd495ab227](https://github.com/cockroachdb/cockroach/commits/0352fbee5e6933e8bca28aae4b136bcd495ab227):


```
test artifacts and logs in: /artifacts/liquibase/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,liquibase.go:123,liquibase.go:140,test_runner.go:896: 
		Tests run on Cockroach v22.2.0-alpha.00000000-2042-g0352fbee5e
		Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
		47 Total Tests Run
		38 tests passed
		9 tests failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
		For a full summary look at the liquibase artifacts 
		
		An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5928734?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5928734?buildTab=artifacts#/liquibase) on master @ [b83e83b7e54fabf6ebd3a42cff67f4b8c5b7b863](https://github.com/cockroachdb/cockroach/commits/b83e83b7e54fabf6ebd3a42cff67f4b8c5b7b863):


```
test artifacts and logs in: /artifacts/liquibase/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,liquibase.go:123,liquibase.go:140,test_runner.go:896: 
		Tests run on Cockroach v22.2.0-alpha.00000000-2049-gb83e83b7e5
		Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
		47 Total Tests Run
		38 tests passed
		9 tests failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
		For a full summary look at the liquibase artifacts 
		
		An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5940811?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5940811?buildTab=artifacts#/liquibase) on master @ [748b89a3fbef294f4b0f930c9dbdf88294b3deeb](https://github.com/cockroachdb/cockroach/commits/748b89a3fbef294f4b0f930c9dbdf88294b3deeb):


```
test artifacts and logs in: /artifacts/liquibase/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,liquibase.go:123,liquibase.go:140,test_runner.go:896: 
		Tests run on Cockroach v22.2.0-alpha.00000000-2096-g748b89a3fb
		Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
		47 Total Tests Run
		38 tests passed
		9 tests failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
		For a full summary look at the liquibase artifacts 
		
		An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5954907?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5954907?buildTab=artifacts#/liquibase) on master @ [7326bbbf94610c40ae2d2fdf42c119dfea93a170](https://github.com/cockroachdb/cockroach/commits/7326bbbf94610c40ae2d2fdf42c119dfea93a170):


```
test artifacts and logs in: /artifacts/liquibase/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,liquibase.go:123,liquibase.go:140,test_runner.go:896: 
		Tests run on Cockroach v22.2.0-alpha.00000000-2143-g7326bbbf94
		Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
		47 Total Tests Run
		38 tests passed
		9 tests failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
		For a full summary look at the liquibase artifacts 
		
		An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5969340?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5969340?buildTab=artifacts#/liquibase) on master @ [4d40b770f96e8248a937b614c5909289c38c777a](https://github.com/cockroachdb/cockroach/commits/4d40b770f96e8248a937b614c5909289c38c777a):


```
test artifacts and logs in: /artifacts/liquibase/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,liquibase.go:123,liquibase.go:140,test_runner.go:896: 
		Tests run on Cockroach v22.2.0-alpha.00000000-2179-g4d40b770f9
		Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
		47 Total Tests Run
		38 tests passed
		9 tests failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
		For a full summary look at the liquibase artifacts 
		
		An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5981014?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5981014?buildTab=artifacts#/liquibase) on master @ [d687b14e513a49f94e1862c78a7685b78f4fbad9](https://github.com/cockroachdb/cockroach/commits/d687b14e513a49f94e1862c78a7685b78f4fbad9):


```
test artifacts and logs in: /artifacts/liquibase/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,liquibase.go:123,liquibase.go:140,test_runner.go:896: 
		Tests run on Cockroach v22.2.0-alpha.00000000-2233-gd687b14e51
		Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
		47 Total Tests Run
		38 tests passed
		9 tests failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
		For a full summary look at the liquibase artifacts 
		
		An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5991660?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5991660?buildTab=artifacts#/liquibase) on master @ [db95c2484a9f15c2de01466af0aae19c8ab57618](https://github.com/cockroachdb/cockroach/commits/db95c2484a9f15c2de01466af0aae19c8ab57618):


```
test artifacts and logs in: /artifacts/liquibase/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,liquibase.go:123,liquibase.go:140,test_runner.go:896: 
		Tests run on Cockroach v22.2.0-alpha.00000000-2259-gdb95c2484a
		Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
		47 Total Tests Run
		38 tests passed
		9 tests failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
		For a full summary look at the liquibase artifacts 
		
		An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5997534?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5997534?buildTab=artifacts#/liquibase) on master @ [a7c91f06d8ee0fa2096bcd626f689009024947bb](https://github.com/cockroachdb/cockroach/commits/a7c91f06d8ee0fa2096bcd626f689009024947bb):


```
test artifacts and logs in: /artifacts/liquibase/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,liquibase.go:123,liquibase.go:140,test_runner.go:896: 
		Tests run on Cockroach v22.2.0-alpha.00000000-2268-ga7c91f06d8
		Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
		47 Total Tests Run
		38 tests passed
		9 tests failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
		For a full summary look at the liquibase artifacts 
		
		An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6004536?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6004536?buildTab=artifacts#/liquibase) on master @ [ce8bfd4811e789bdd4cf31640722517a5cf46237](https://github.com/cockroachdb/cockroach/commits/ce8bfd4811e789bdd4cf31640722517a5cf46237):


```
test artifacts and logs in: /artifacts/liquibase/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,liquibase.go:123,liquibase.go:140,test_runner.go:896: 
		Tests run on Cockroach v22.2.0-alpha.00000000-2273-gce8bfd4811
		Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
		47 Total Tests Run
		38 tests passed
		9 tests failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
		For a full summary look at the liquibase artifacts 
		
		An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6019684?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6019684?buildTab=artifacts#/liquibase) on master @ [9a0a233f4021f25c1766434bb6aee85536e32036](https://github.com/cockroachdb/cockroach/commits/9a0a233f4021f25c1766434bb6aee85536e32036):


```
test artifacts and logs in: /artifacts/liquibase/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,liquibase.go:123,liquibase.go:140,test_runner.go:896: 
		Tests run on Cockroach v22.2.0-alpha.00000000-2326-g9a0a233f40
		Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
		47 Total Tests Run
		38 tests passed
		9 tests failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
		For a full summary look at the liquibase artifacts 
		
		An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6034844?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6034844?buildTab=artifacts#/liquibase) on master @ [f53c21f9fe0fe07ba32d2f28ac4bd2c4bc2ed50b](https://github.com/cockroachdb/cockroach/commits/f53c21f9fe0fe07ba32d2f28ac4bd2c4bc2ed50b):


```
test artifacts and logs in: /artifacts/liquibase/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,liquibase.go:123,liquibase.go:140,test_runner.go:896: 
		Tests run on Cockroach v22.2.0-alpha.00000000-2390-gf53c21f9fe
		Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
		47 Total Tests Run
		38 tests passed
		9 tests failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
		For a full summary look at the liquibase artifacts 
		
		An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6048219?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6048219?buildTab=artifacts#/liquibase) on master @ [43bfc3185cb362b14e0b8a331148f0b14542a4f2](https://github.com/cockroachdb/cockroach/commits/43bfc3185cb362b14e0b8a331148f0b14542a4f2):


```
test artifacts and logs in: /artifacts/liquibase/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,liquibase.go:123,liquibase.go:140,test_runner.go:896: 
		Tests run on Cockroach v22.2.0-alpha.00000000-2434-g43bfc3185c
		Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
		47 Total Tests Run
		38 tests passed
		9 tests failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
		For a full summary look at the liquibase artifacts 
		
		An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6064717?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6064717?buildTab=artifacts#/liquibase) on master @ [773f7d4445ce3e0e806b7a182adba70a0f270f19](https://github.com/cockroachdb/cockroach/commits/773f7d4445ce3e0e806b7a182adba70a0f270f19):


```
test artifacts and logs in: /artifacts/liquibase/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,liquibase.go:123,liquibase.go:140,test_runner.go:896: 
		Tests run on Cockroach v22.2.0-alpha.00000000-2493-g773f7d4445
		Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
		47 Total Tests Run
		38 tests passed
		9 tests failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
		For a full summary look at the liquibase artifacts 
		
		An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6084542?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6084542?buildTab=artifacts#/liquibase) on master @ [0dd438d3dc0b42543890455945a7a6b42811def1](https://github.com/cockroachdb/cockroach/commits/0dd438d3dc0b42543890455945a7a6b42811def1):


```
test artifacts and logs in: /artifacts/liquibase/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,liquibase.go:123,liquibase.go:140,test_runner.go:896: 
		Tests run on Cockroach v22.2.0-alpha.00000000-2573-g0dd438d3dc
		Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
		47 Total Tests Run
		38 tests passed
		9 tests failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
		For a full summary look at the liquibase artifacts 
		
		An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6087204?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6087204?buildTab=artifacts#/liquibase) on master @ [d25cb57ccd9bc643ce9058ebd2057cab36b69ad5](https://github.com/cockroachdb/cockroach/commits/d25cb57ccd9bc643ce9058ebd2057cab36b69ad5):


```
test artifacts and logs in: /artifacts/liquibase/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,liquibase.go:123,liquibase.go:140,test_runner.go:896: 
		Tests run on Cockroach v22.2.0-alpha.00000000-2622-gd25cb57ccd
		Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
		47 Total Tests Run
		38 tests passed
		9 tests failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
		For a full summary look at the liquibase artifacts 
		
		An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6094282?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6094282?buildTab=artifacts#/liquibase) on master @ [41db784cb97d2749b162020c2c821979094f87b1](https://github.com/cockroachdb/cockroach/commits/41db784cb97d2749b162020c2c821979094f87b1):


```
test artifacts and logs in: /artifacts/liquibase/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,liquibase.go:123,liquibase.go:140,test_runner.go:896: 
		Tests run on Cockroach v22.2.0-alpha.00000000-2646-g41db784cb9
		Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
		47 Total Tests Run
		38 tests passed
		9 tests failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
		For a full summary look at the liquibase artifacts 
		
		An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6107800?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6107800?buildTab=artifacts#/liquibase) on master @ [f4042d47fa8062a612c38d4696eb6bee9cee7c21](https://github.com/cockroachdb/cockroach/commits/f4042d47fa8062a612c38d4696eb6bee9cee7c21):


```
test artifacts and logs in: /artifacts/liquibase/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,liquibase.go:123,liquibase.go:140,test_runner.go:896: 
		Tests run on Cockroach v22.2.0-alpha.00000000-2689-gf4042d47fa
		Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
		47 Total Tests Run
		38 tests passed
		9 tests failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
		For a full summary look at the liquibase artifacts 
		
		An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6124150?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6124150?buildTab=artifacts#/liquibase) on master @ [b173a16715e71e94115820374da1eb350b3b459d](https://github.com/cockroachdb/cockroach/commits/b173a16715e71e94115820374da1eb350b3b459d):


```
test artifacts and logs in: /artifacts/liquibase/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,liquibase.go:123,liquibase.go:140,test_runner.go:896: 
		Tests run on Cockroach v22.2.0-alpha.00000000-2750-gb173a16715
		Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
		47 Total Tests Run
		38 tests passed
		9 tests failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
		For a full summary look at the liquibase artifacts 
		
		An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6137147?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6137147?buildTab=artifacts#/liquibase) on master @ [5c2c62ecf1bea60c807edc6b4da22d900ad4ae03](https://github.com/cockroachdb/cockroach/commits/5c2c62ecf1bea60c807edc6b4da22d900ad4ae03):


```
test artifacts and logs in: /artifacts/liquibase/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,liquibase.go:123,liquibase.go:140,test_runner.go:896: 
		Tests run on Cockroach v22.2.0-alpha.00000000-2800-g5c2c62ecf1
		Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
		47 Total Tests Run
		38 tests passed
		9 tests failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
		For a full summary look at the liquibase artifacts 
		
		An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6151881?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6151881?buildTab=artifacts#/liquibase) on master @ [a0d8839aa6164af81a9ebb140147d3baf5321287](https://github.com/cockroachdb/cockroach/commits/a0d8839aa6164af81a9ebb140147d3baf5321287):


```
test artifacts and logs in: /artifacts/liquibase/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,liquibase.go:123,liquibase.go:140,test_runner.go:896: 
		Tests run on Cockroach v22.2.0-alpha.00000000-2834-ga0d8839aa6
		Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
		47 Total Tests Run
		38 tests passed
		9 tests failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
		For a full summary look at the liquibase artifacts 
		
		An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6166285?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6166285?buildTab=artifacts#/liquibase) on master @ [cb55144cdec54d2a70f074ad64b4eca5e6c6891a](https://github.com/cockroachdb/cockroach/commits/cb55144cdec54d2a70f074ad64b4eca5e6c6891a):


```
test artifacts and logs in: /artifacts/liquibase/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,liquibase.go:123,liquibase.go:140,test_runner.go:896: 
		Tests run on Cockroach v22.2.0-alpha.00000000-2899-gcb55144cde
		Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
		47 Total Tests Run
		38 tests passed
		9 tests failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
		For a full summary look at the liquibase artifacts 
		
		An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6174190?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6174190?buildTab=artifacts#/liquibase) on master @ [e6a7dc2f8ee39549e186bd05626c4c375b76fd04](https://github.com/cockroachdb/cockroach/commits/e6a7dc2f8ee39549e186bd05626c4c375b76fd04):


```
test artifacts and logs in: /artifacts/liquibase/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,liquibase.go:123,liquibase.go:140,test_runner.go:896: 
		Tests run on Cockroach v22.2.0-alpha.00000000-2938-ge6a7dc2f8e
		Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
		47 Total Tests Run
		38 tests passed
		9 tests failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
		For a full summary look at the liquibase artifacts 
		
		An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6181118?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6181118?buildTab=artifacts#/liquibase) on master @ [aaf50e920ceff3c2863ab96b9e3614b8434b70a8](https://github.com/cockroachdb/cockroach/commits/aaf50e920ceff3c2863ab96b9e3614b8434b70a8):


```
test artifacts and logs in: /artifacts/liquibase/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,liquibase.go:123,liquibase.go:140,test_runner.go:896: 
		Tests run on Cockroach v22.2.0-alpha.00000000-2943-gaaf50e920c
		Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
		47 Total Tests Run
		38 tests passed
		9 tests failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
		For a full summary look at the liquibase artifacts 
		
		An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6195961?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6195961?buildTab=artifacts#/liquibase) on master @ [80c274877a917580af62be6eb0cd48c8c7ae9c08](https://github.com/cockroachdb/cockroach/commits/80c274877a917580af62be6eb0cd48c8c7ae9c08):


```
test artifacts and logs in: /artifacts/liquibase/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,liquibase.go:123,liquibase.go:140,test_runner.go:896: 
		Tests run on Cockroach v22.2.0-alpha.00000000-3003-g80c274877a
		Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
		47 Total Tests Run
		38 tests passed
		9 tests failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
		For a full summary look at the liquibase artifacts 
		
		An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6213710?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6213710?buildTab=artifacts#/liquibase) on master @ [003c0360de8b64319b5f0f127b99be91dbdca8a3](https://github.com/cockroachdb/cockroach/commits/003c0360de8b64319b5f0f127b99be91dbdca8a3):


```
test artifacts and logs in: /artifacts/liquibase/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,liquibase.go:123,liquibase.go:140,test_runner.go:896: 
		Tests run on Cockroach v22.2.0-alpha.00000000-3029-g003c0360de
		Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
		47 Total Tests Run
		38 tests passed
		9 tests failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
		For a full summary look at the liquibase artifacts 
		
		An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6231545?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6231545?buildTab=artifacts#/liquibase) on master @ [524fd14da3fefcd849f44a835cc5f88f5dbdadcc](https://github.com/cockroachdb/cockroach/commits/524fd14da3fefcd849f44a835cc5f88f5dbdadcc):


```
test artifacts and logs in: /artifacts/liquibase/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,liquibase.go:123,liquibase.go:140,test_runner.go:896: 
		Tests run on Cockroach v22.2.0-alpha.00000000-3077-g524fd14da3
		Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
		47 Total Tests Run
		38 tests passed
		9 tests failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
		For a full summary look at the liquibase artifacts 
		
		An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6260325?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6260325?buildTab=artifacts#/liquibase) on master @ [770ff3c545a51752490403da64d56fb397f49c5e](https://github.com/cockroachdb/cockroach/commits/770ff3c545a51752490403da64d56fb397f49c5e):


```
test artifacts and logs in: /artifacts/liquibase/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,liquibase.go:123,liquibase.go:140,test_runner.go:897: 
		Tests run on Cockroach v22.2.0-alpha.00000000-3201-g770ff3c545
		Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
		47 Total Tests Run
		38 tests passed
		9 tests failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
		For a full summary look at the liquibase artifacts 
		
		An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6267021?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6267021?buildTab=artifacts#/liquibase) on master @ [f59620ec646d1181d358d0dc41ab60815ecf59c9](https://github.com/cockroachdb/cockroach/commits/f59620ec646d1181d358d0dc41ab60815ecf59c9):


```
test artifacts and logs in: /artifacts/liquibase/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,liquibase.go:123,liquibase.go:140,test_runner.go:897: 
		Tests run on Cockroach v22.2.0-alpha.00000000-3212-gf59620ec64
		Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
		47 Total Tests Run
		38 tests passed
		9 tests failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
		For a full summary look at the liquibase artifacts 
		
		An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6273143?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6273143?buildTab=artifacts#/liquibase) on master @ [3b16435371a43d603d193a1e2b480a23fba3f07a](https://github.com/cockroachdb/cockroach/commits/3b16435371a43d603d193a1e2b480a23fba3f07a):


```
test artifacts and logs in: /artifacts/liquibase/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,liquibase.go:123,liquibase.go:140,test_runner.go:897: 
		Tests run on Cockroach v22.2.0-alpha.00000000-3220-g3b16435371
		Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
		47 Total Tests Run
		38 tests passed
		9 tests failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
		For a full summary look at the liquibase artifacts 
		
		An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6287501?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6287501?buildTab=artifacts#/liquibase) on master @ [d35c174b71065264b8a3910df3f488d10741f788](https://github.com/cockroachdb/cockroach/commits/d35c174b71065264b8a3910df3f488d10741f788):


```
test artifacts and logs in: /artifacts/liquibase/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,liquibase.go:123,liquibase.go:140,test_runner.go:897: 
		Tests run on Cockroach v22.2.0-alpha.00000000-3264-gd35c174b71
		Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
		47 Total Tests Run
		38 tests passed
		9 tests failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
		For a full summary look at the liquibase artifacts 
		
		An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6300492?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6300492?buildTab=artifacts#/liquibase) on master @ [448352ce4ed71a58e7701e82c786dd5152498310](https://github.com/cockroachdb/cockroach/commits/448352ce4ed71a58e7701e82c786dd5152498310):


```
test artifacts and logs in: /artifacts/liquibase/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,liquibase.go:123,liquibase.go:140,test_runner.go:897: 
		Tests run on Cockroach v22.2.0-alpha.1-172-g448352ce4e
		Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
		47 Total Tests Run
		38 tests passed
		9 tests failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
		For a full summary look at the liquibase artifacts 
		
		An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6315043?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6315043?buildTab=artifacts#/liquibase) on master @ [b316a5ed5fe7253d113174d9d95ddebf1143b4e4](https://github.com/cockroachdb/cockroach/commits/b316a5ed5fe7253d113174d9d95ddebf1143b4e4):


```
test artifacts and logs in: /artifacts/liquibase/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,liquibase.go:123,liquibase.go:140,test_runner.go:897: 
		Tests run on Cockroach v22.2.0-alpha.1-239-gb316a5ed5f
		Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
		47 Total Tests Run
		38 tests passed
		9 tests failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
		For a full summary look at the liquibase artifacts 
		
		An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6329079?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6329079?buildTab=artifacts#/liquibase) on master @ [c080f7ac2f5be11aedd787dceba77cca7df18a16](https://github.com/cockroachdb/cockroach/commits/c080f7ac2f5be11aedd787dceba77cca7df18a16):


```
test artifacts and logs in: /artifacts/liquibase/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,liquibase.go:123,liquibase.go:140,test_runner.go:897: 
		Tests run on Cockroach v22.2.0-alpha.1-293-gc080f7ac2f
		Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
		47 Total Tests Run
		38 tests passed
		9 tests failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
		For a full summary look at the liquibase artifacts 
		
		An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6336049?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6336049?buildTab=artifacts#/liquibase) on master @ [4dcb32c0346e20a95847763f89b9b0796d9ed4dc](https://github.com/cockroachdb/cockroach/commits/4dcb32c0346e20a95847763f89b9b0796d9ed4dc):


```
test artifacts and logs in: /artifacts/liquibase/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,liquibase.go:123,liquibase.go:140,test_runner.go:897: 
		Tests run on Cockroach v22.2.0-alpha.1-303-g4dcb32c034
		Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
		47 Total Tests Run
		38 tests passed
		9 tests failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
		For a full summary look at the liquibase artifacts 
		
		An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6341855?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6341855?buildTab=artifacts#/liquibase) on master @ [b0f13c9bbef3e6628471d887672be7c7658f6511](https://github.com/cockroachdb/cockroach/commits/b0f13c9bbef3e6628471d887672be7c7658f6511):


```
test artifacts and logs in: /artifacts/liquibase/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,liquibase.go:123,liquibase.go:140,test_runner.go:897: 
		Tests run on Cockroach v22.2.0-alpha.1-305-gb0f13c9bbe
		Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
		47 Total Tests Run
		38 tests passed
		9 tests failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
		For a full summary look at the liquibase artifacts 
		
		An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6347309?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6347309?buildTab=artifacts#/liquibase) on master @ [e5917605f751c569ddd90062250a40cafd7ccf30](https://github.com/cockroachdb/cockroach/commits/e5917605f751c569ddd90062250a40cafd7ccf30):


```
test artifacts and logs in: /artifacts/liquibase/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,liquibase.go:123,liquibase.go:140,test_runner.go:897: 
		Tests run on Cockroach v22.2.0-alpha.1-309-ge5917605f7
		Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
		47 Total Tests Run
		38 tests passed
		9 tests failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
		For a full summary look at the liquibase artifacts 
		
		An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6354744?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6354744?buildTab=artifacts#/liquibase) on master @ [6ffaa1d907b7c9194df4ffff412f48a63bf432df](https://github.com/cockroachdb/cockroach/commits/6ffaa1d907b7c9194df4ffff412f48a63bf432df):


```
test artifacts and logs in: /artifacts/liquibase/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,liquibase.go:123,liquibase.go:140,test_runner.go:897: 
		Tests run on Cockroach v22.2.0-alpha.1-318-g6ffaa1d907
		Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
		47 Total Tests Run
		38 tests passed
		9 tests failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
		For a full summary look at the liquibase artifacts 
		
		An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6369513?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6369513?buildTab=artifacts#/liquibase) on master @ [2372698da1dfacb90f60c6a63f2c1298d1db16b8](https://github.com/cockroachdb/cockroach/commits/2372698da1dfacb90f60c6a63f2c1298d1db16b8):


```
test artifacts and logs in: /artifacts/liquibase/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,liquibase.go:123,liquibase.go:140,test_runner.go:897: 
		Tests run on Cockroach v22.2.0-alpha.2-246-g2372698da1
		Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
		47 Total Tests Run
		38 tests passed
		9 tests failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
		For a full summary look at the liquibase artifacts 
		
		An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6383257?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6383257?buildTab=artifacts#/liquibase) on master @ [e39111b2e714375faa0facc05e51e8f619a55b21](https://github.com/cockroachdb/cockroach/commits/e39111b2e714375faa0facc05e51e8f619a55b21):


```
test artifacts and logs in: /artifacts/liquibase/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,liquibase.go:123,liquibase.go:140,test_runner.go:908: 
		Tests run on Cockroach v22.2.0-alpha.2-305-ge39111b2e7
		Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
		47 Total Tests Run
		38 tests passed
		9 tests failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
		For a full summary look at the liquibase artifacts 
		
		An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #87586 roachtest: liquibase failed [C-test-failure O-roachtest O-robot branch-release-22.2 release-blocker]
- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6400376?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6400376?buildTab=artifacts#/liquibase) on master @ [a82711442c65cf14489c55041b45b11a1e38415b](https://github.com/cockroachdb/cockroach/commits/a82711442c65cf14489c55041b45b11a1e38415b):


```
test artifacts and logs in: /artifacts/liquibase/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,liquibase.go:123,liquibase.go:140,test_runner.go:906: 
		Tests run on Cockroach v22.2.0-alpha.2-371-ga82711442c
		Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
		47 Total Tests Run
		38 tests passed
		9 tests failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
		For a full summary look at the liquibase artifacts 
		
		An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #87586 roachtest: liquibase failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6415593?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6415593?buildTab=artifacts#/liquibase) on master @ [389661e823c19f318fa07ec2278336262531692d](https://github.com/cockroachdb/cockroach/commits/389661e823c19f318fa07ec2278336262531692d):


```
test artifacts and logs in: /artifacts/liquibase/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,liquibase.go:123,liquibase.go:140,test_runner.go:917: 
		Tests run on Cockroach v22.2.0-alpha.2-430-g389661e823
		Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
		47 Total Tests Run
		38 tests passed
		9 tests failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
		For a full summary look at the liquibase artifacts 
		
		An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #87586 roachtest: liquibase failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6423857?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6423857?buildTab=artifacts#/liquibase) on master @ [bc2e47da0523b347c28cf024707e80cd35d6c98a](https://github.com/cockroachdb/cockroach/commits/bc2e47da0523b347c28cf024707e80cd35d6c98a):


```
test artifacts and logs in: /artifacts/liquibase/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,liquibase.go:123,liquibase.go:140,test_runner.go:917: 
		Tests run on Cockroach v22.2.0-alpha.2-432-gbc2e47da05
		Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
		47 Total Tests Run
		38 tests passed
		9 tests failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
		For a full summary look at the liquibase artifacts 
		
		An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #87586 roachtest: liquibase failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6433129?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6433129?buildTab=artifacts#/liquibase) on master @ [773568fbda06ba9be9fb1bc34a331f21c8891ffa](https://github.com/cockroachdb/cockroach/commits/773568fbda06ba9be9fb1bc34a331f21c8891ffa):


```
test artifacts and logs in: /artifacts/liquibase/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,liquibase.go:123,liquibase.go:140,test_runner.go:917: 
		Tests run on Cockroach v22.2.0-alpha.2-439-g773568fbda
		Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
		47 Total Tests Run
		38 tests passed
		9 tests failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
		For a full summary look at the liquibase artifacts 
		
		An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #87586 roachtest: liquibase failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6448808?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6448808?buildTab=artifacts#/liquibase) on master @ [2cfa6b1779ead01508c04e228bf72b4e0e96d98c](https://github.com/cockroachdb/cockroach/commits/2cfa6b1779ead01508c04e228bf72b4e0e96d98c):


```
test artifacts and logs in: /artifacts/liquibase/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,liquibase.go:123,liquibase.go:140,test_runner.go:917: 
		Tests run on Cockroach v22.2.0-alpha.3-228-g2cfa6b1779
		Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
		47 Total Tests Run
		38 tests passed
		9 tests failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
		For a full summary look at the liquibase artifacts 
		
		An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #87586 roachtest: liquibase failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6460155?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6460155?buildTab=artifacts#/liquibase) on master @ [95677eb5f8d006629b16024fb7d87d55344c1470](https://github.com/cockroachdb/cockroach/commits/95677eb5f8d006629b16024fb7d87d55344c1470):


```
test artifacts and logs in: /artifacts/liquibase/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,liquibase.go:123,liquibase.go:140,test_runner.go:917: 
		Tests run on Cockroach v22.2.0-alpha.3-245-g95677eb5f8
		Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
		47 Total Tests Run
		38 tests passed
		9 tests failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
		For a full summary look at the liquibase artifacts 
		
		An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #87586 roachtest: liquibase failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6471013?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6471013?buildTab=artifacts#/liquibase) on master @ [1f24e52aea4195ffe26b97160079bb29f264338a](https://github.com/cockroachdb/cockroach/commits/1f24e52aea4195ffe26b97160079bb29f264338a):


```
test artifacts and logs in: /artifacts/liquibase/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,liquibase.go:123,liquibase.go:140,test_runner.go:917: 
		Tests run on Cockroach v22.2.0-alpha.3-264-g1f24e52aea
		Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
		47 Total Tests Run
		38 tests passed
		9 tests failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
		For a full summary look at the liquibase artifacts 
		
		An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #87586 roachtest: liquibase failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6482566?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6482566?buildTab=artifacts#/liquibase) on master @ [9a05046ce19e7678340e82c70d61e928be95bc72](https://github.com/cockroachdb/cockroach/commits/9a05046ce19e7678340e82c70d61e928be95bc72):


```
test artifacts and logs in: /artifacts/liquibase/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,liquibase.go:123,liquibase.go:140,test_runner.go:917: 
		Tests run on Cockroach v22.2.0-alpha.3-291-g9a05046ce1
		Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
		47 Total Tests Run
		38 tests passed
		9 tests failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
		For a full summary look at the liquibase artifacts 
		
		An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #87586 roachtest: liquibase failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6495696?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6495696?buildTab=artifacts#/liquibase) on master @ [bd97ad5b8c9f537a89492a051574d867469bef33](https://github.com/cockroachdb/cockroach/commits/bd97ad5b8c9f537a89492a051574d867469bef33):


```
test artifacts and logs in: /artifacts/liquibase/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,liquibase.go:123,liquibase.go:140,test_runner.go:917: 
		Tests run on Cockroach v22.2.0-alpha.3-312-gbd97ad5b8c
		Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
		47 Total Tests Run
		38 tests passed
		9 tests failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
		For a full summary look at the liquibase artifacts 
		
		An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #87586 roachtest: liquibase failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6504163?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6504163?buildTab=artifacts#/liquibase) on master @ [3fa1a1600898d7b78b9e39d07132a387a2f9a1b6](https://github.com/cockroachdb/cockroach/commits/3fa1a1600898d7b78b9e39d07132a387a2f9a1b6):


```
test artifacts and logs in: /artifacts/liquibase/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,liquibase.go:123,liquibase.go:140,test_runner.go:917: 
		Tests run on Cockroach v22.2.0-alpha.3-315-g3fa1a16008
		Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
		47 Total Tests Run
		38 tests passed
		9 tests failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
		For a full summary look at the liquibase artifacts 
		
		An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #87586 roachtest: liquibase failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6513239?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6513239?buildTab=artifacts#/liquibase) on master @ [87ed064dc23eab6948ee8a07e8507f150bda0e44](https://github.com/cockroachdb/cockroach/commits/87ed064dc23eab6948ee8a07e8507f150bda0e44):


```
test artifacts and logs in: /artifacts/liquibase/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,liquibase.go:123,liquibase.go:140,test_runner.go:917: 
		Tests run on Cockroach v22.2.0-alpha.3-323-g87ed064dc2
		Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
		47 Total Tests Run
		38 tests passed
		9 tests failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
		For a full summary look at the liquibase artifacts 
		
		An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #87586 roachtest: liquibase failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6530764?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6530764?buildTab=artifacts#/liquibase) on master @ [5862e9b3c1e24b3f644716789c96108a8e92fd71](https://github.com/cockroachdb/cockroach/commits/5862e9b3c1e24b3f644716789c96108a8e92fd71):


```
test artifacts and logs in: /artifacts/liquibase/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,liquibase.go:123,liquibase.go:140,test_runner.go:917: 
		Tests run on Cockroach v22.2.0-alpha.3-374-g5862e9b3c1
		Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
		47 Total Tests Run
		38 tests passed
		9 tests failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
		For a full summary look at the liquibase artifacts 
		
		An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #87586 roachtest: liquibase failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6549169?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6549169?buildTab=artifacts#/liquibase) on master @ [bbcb10f982e47943de63a3a2d1faf10d3eb49e75](https://github.com/cockroachdb/cockroach/commits/bbcb10f982e47943de63a3a2d1faf10d3eb49e75):


```
test artifacts and logs in: /artifacts/liquibase/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,liquibase.go:123,liquibase.go:140,test_runner.go:928: 
		Tests run on Cockroach v22.2.0-alpha.3-430-gbbcb10f982
		Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
		47 Total Tests Run
		38 tests passed
		9 tests failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
		For a full summary look at the liquibase artifacts 
		
		An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #87586 roachtest: liquibase failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6567232?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6567232?buildTab=artifacts#/liquibase) on master @ [726cf22b9f06b766d857b4617dec0df18d1e5cd0](https://github.com/cockroachdb/cockroach/commits/726cf22b9f06b766d857b4617dec0df18d1e5cd0):


```
test artifacts and logs in: /artifacts/liquibase/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,liquibase.go:123,liquibase.go:140,test_runner.go:928: 
		Tests run on Cockroach v22.2.0-alpha.3-472-g726cf22b9f
		Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
		47 Total Tests Run
		38 tests passed
		9 tests failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
		For a full summary look at the liquibase artifacts 
		
		An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #87586 roachtest: liquibase failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6584253?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6584253?buildTab=artifacts#/liquibase) on master @ [34dc56fbb5789b39be47b110bf22332c7f5654f6](https://github.com/cockroachdb/cockroach/commits/34dc56fbb5789b39be47b110bf22332c7f5654f6):


```
test artifacts and logs in: /artifacts/liquibase/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,liquibase.go:123,liquibase.go:140,test_runner.go:928: 
		Tests run on Cockroach v22.2.0-alpha.3-527-g34dc56fbb5
		Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
		47 Total Tests Run
		38 tests passed
		9 tests failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
		For a full summary look at the liquibase artifacts 
		
		An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #87586 roachtest: liquibase failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6601244?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6601244?buildTab=artifacts#/liquibase) on master @ [89f4ad907a1756551bd6864c3e8516eeff6b0e0a](https://github.com/cockroachdb/cockroach/commits/89f4ad907a1756551bd6864c3e8516eeff6b0e0a):


```
test artifacts and logs in: /artifacts/liquibase/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,liquibase.go:123,liquibase.go:140,test_runner.go:928: 
		Tests run on Cockroach v22.2.0-alpha.3-581-g89f4ad907a
		Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
		47 Total Tests Run
		38 tests passed
		9 tests failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
		For a full summary look at the liquibase artifacts 
		
		An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #87586 roachtest: liquibase failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6610307?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6610307?buildTab=artifacts#/liquibase) on master @ [51c8aae748d338549400c047796c6c9b892527da](https://github.com/cockroachdb/cockroach/commits/51c8aae748d338549400c047796c6c9b892527da):


```
test artifacts and logs in: /artifacts/liquibase/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,liquibase.go:123,liquibase.go:140,test_runner.go:928: 
		Tests run on Cockroach v22.2.0-alpha.3-583-g51c8aae748
		Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
		47 Total Tests Run
		38 tests passed
		9 tests failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
		For a full summary look at the liquibase artifacts 
		
		An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #87586 roachtest: liquibase failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6619262?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6619262?buildTab=artifacts#/liquibase) on master @ [ff13325e9368c4e8dd9a4d5cf4aa2ad2f33e9ac0](https://github.com/cockroachdb/cockroach/commits/ff13325e9368c4e8dd9a4d5cf4aa2ad2f33e9ac0):


```
test artifacts and logs in: /artifacts/liquibase/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,liquibase.go:123,liquibase.go:140,test_runner.go:928: 
		Tests run on Cockroach v22.2.0-alpha.3-593-gff13325e93
		Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
		47 Total Tests Run
		38 tests passed
		9 tests failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
		For a full summary look at the liquibase artifacts 
		
		An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #87586 roachtest: liquibase failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6628566?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6628566?buildTab=artifacts#/liquibase) on master @ [ff13325e9368c4e8dd9a4d5cf4aa2ad2f33e9ac0](https://github.com/cockroachdb/cockroach/commits/ff13325e9368c4e8dd9a4d5cf4aa2ad2f33e9ac0):


```
test artifacts and logs in: /artifacts/liquibase/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,liquibase.go:123,liquibase.go:140,test_runner.go:928: 
		Tests run on Cockroach v22.2.0-alpha.3-593-gff13325e93
		Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
		47 Total Tests Run
		38 tests passed
		9 tests failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
		For a full summary look at the liquibase artifacts 
		
		An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #87586 roachtest: liquibase failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6661603?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6661603?buildTab=artifacts#/liquibase) on master @ [4982322e91c37c6d0249dc5597b9e02eec19848d](https://github.com/cockroachdb/cockroach/commits/4982322e91c37c6d0249dc5597b9e02eec19848d):


```
test artifacts and logs in: /artifacts/liquibase/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,liquibase.go:123,liquibase.go:140,test_runner.go:928: 
		Tests run on Cockroach v22.2.0-alpha.3-690-g4982322e91
		Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
		47 Total Tests Run
		38 tests passed
		9 tests failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
		For a full summary look at the liquibase artifacts 
		
		An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #87586 roachtest: liquibase failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6683300?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6683300?buildTab=artifacts#/liquibase) on master @ [4780ee15194e7d22b661ac92254b13cd2b71dcb1](https://github.com/cockroachdb/cockroach/commits/4780ee15194e7d22b661ac92254b13cd2b71dcb1):


```
test artifacts and logs in: /artifacts/liquibase/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,liquibase.go:123,liquibase.go:140,test_runner.go:928: 
		Tests run on Cockroach v22.2.0-alpha.3-734-g4780ee1519
		Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
		47 Total Tests Run
		38 tests passed
		9 tests failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
		For a full summary look at the liquibase artifacts 
		
		An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #87586 roachtest: liquibase failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6698314?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6698314?buildTab=artifacts#/liquibase) on master @ [4780ee15194e7d22b661ac92254b13cd2b71dcb1](https://github.com/cockroachdb/cockroach/commits/4780ee15194e7d22b661ac92254b13cd2b71dcb1):


```
test artifacts and logs in: /artifacts/liquibase/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,liquibase.go:123,liquibase.go:140,test_runner.go:928: 
		Tests run on Cockroach v22.2.0-alpha.3-734-g4780ee1519
		Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
		47 Total Tests Run
		38 tests passed
		9 tests failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
		For a full summary look at the liquibase artifacts 
		
		An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #87586 roachtest: liquibase failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6704781?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6704781?buildTab=artifacts#/liquibase) on master @ [801bfc62afd7128be180e3396d21a1e0b2daa227](https://github.com/cockroachdb/cockroach/commits/801bfc62afd7128be180e3396d21a1e0b2daa227):


```
test artifacts and logs in: /artifacts/liquibase/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,liquibase.go:123,liquibase.go:140,test_runner.go:928: 
		Tests run on Cockroach v22.2.0-alpha.3-755-g801bfc62af
		Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
		47 Total Tests Run
		38 tests passed
		9 tests failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
		For a full summary look at the liquibase artifacts 
		
		An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #87586 roachtest: liquibase failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6731749?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6731749?buildTab=artifacts#/liquibase) on master @ [e786cba2b137a671d3846cf7a33e7b9dea2854e6](https://github.com/cockroachdb/cockroach/commits/e786cba2b137a671d3846cf7a33e7b9dea2854e6):


```
test artifacts and logs in: /artifacts/liquibase/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,liquibase.go:123,liquibase.go:140,test_runner.go:928: 
		Tests run on Cockroach v22.2.0-alpha.3-814-ge786cba2b1
		Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
		47 Total Tests Run
		38 tests passed
		9 tests failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
		For a full summary look at the liquibase artifacts 
		
		An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #87586 roachtest: liquibase failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6741071?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6741071?buildTab=artifacts#/liquibase) on master @ [ff6961ef1741766b5cbe0b64ebf754e82343eb00](https://github.com/cockroachdb/cockroach/commits/ff6961ef1741766b5cbe0b64ebf754e82343eb00):


```
test artifacts and logs in: /artifacts/liquibase/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,liquibase.go:123,liquibase.go:140,test_runner.go:928: 
		Tests run on Cockroach v22.2.0-alpha.3-829-gff6961ef17
		Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
		47 Total Tests Run
		38 tests passed
		9 tests failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
		For a full summary look at the liquibase artifacts 
		
		An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #87586 roachtest: liquibase failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6749797?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6749797?buildTab=artifacts#/liquibase) on master @ [84384b50c023dd4c05fff76af85a6975f5d2b0ab](https://github.com/cockroachdb/cockroach/commits/84384b50c023dd4c05fff76af85a6975f5d2b0ab):


```
test artifacts and logs in: /artifacts/liquibase/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,liquibase.go:123,liquibase.go:140,test_runner.go:928: 
		Tests run on Cockroach v22.2.0-alpha.3-835-g84384b50c0
		Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
		47 Total Tests Run
		38 tests passed
		9 tests failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
		For a full summary look at the liquibase artifacts 
		
		An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #87586 roachtest: liquibase failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6775433?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6775433?buildTab=artifacts#/liquibase) on master @ [0eaeeb773474716753781289788fdd087fb9b166](https://github.com/cockroachdb/cockroach/commits/0eaeeb773474716753781289788fdd087fb9b166):


```
test artifacts and logs in: /artifacts/liquibase/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,liquibase.go:123,liquibase.go:140,test_runner.go:928: 
		Tests run on Cockroach v22.2.0-alpha.3-887-g0eaeeb7734
		Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
		47 Total Tests Run
		38 tests passed
		9 tests failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
		--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
		For a full summary look at the liquibase artifacts 
		
		An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #87586 roachtest: liquibase failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6784051?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6784051?buildTab=artifacts#/liquibase) on master @ [d090ac9e42263b3fe8ec94d156bed056d06bedac](https://github.com/cockroachdb/cockroach/commits/d090ac9e42263b3fe8ec94d156bed056d06bedac):


```
test artifacts and logs in: /artifacts/liquibase/run_1
(orm_helpers.go:191).summarizeFailed: 
Tests run on Cockroach v22.2.0-alpha.3-929-gd090ac9e42
Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
47 Total Tests Run
38 tests passed
9 tests failed
17 tests skipped
0 tests ignored
0 tests passed unexpectedly
6 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
For a full summary look at the liquibase artifacts
An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89368 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0 release-blocker]
- #87586 roachtest: liquibase failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6803698?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6803698?buildTab=artifacts#/liquibase) on master @ [1f7d3baebdee811380b4f3e8a0d634701041c5d0](https://github.com/cockroachdb/cockroach/commits/1f7d3baebdee811380b4f3e8a0d634701041c5d0):


```
test artifacts and logs in: /artifacts/liquibase/run_1
(orm_helpers.go:191).summarizeFailed: 
Tests run on Cockroach v22.2.0-alpha.3-960-g1f7d3baebd
Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
47 Total Tests Run
38 tests passed
9 tests failed
17 tests skipped
0 tests ignored
0 tests passed unexpectedly
6 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
For a full summary look at the liquibase artifacts
An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89368 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #87586 roachtest: liquibase failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6824043?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6824043?buildTab=artifacts#/liquibase) on master @ [ded91681f2be8816238a05850170bde7f61b209f](https://github.com/cockroachdb/cockroach/commits/ded91681f2be8816238a05850170bde7f61b209f):


```
test artifacts and logs in: /artifacts/liquibase/run_1
(orm_helpers.go:191).summarizeFailed: 
Tests run on Cockroach v22.2.0-alpha.3-1010-gded91681f2
Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
47 Total Tests Run
38 tests passed
9 tests failed
17 tests skipped
0 tests ignored
0 tests passed unexpectedly
6 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
For a full summary look at the liquibase artifacts
An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89368 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #87586 roachtest: liquibase failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6837129?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6837129?buildTab=artifacts#/liquibase) on master @ [e06d2286b011096526eda7f2d7f7bb7acea0ae84](https://github.com/cockroachdb/cockroach/commits/e06d2286b011096526eda7f2d7f7bb7acea0ae84):


```
test artifacts and logs in: /artifacts/liquibase/run_1
(orm_helpers.go:191).summarizeFailed: 
Tests run on Cockroach v22.2.0-alpha.3-1023-ge06d2286b0
Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
47 Total Tests Run
38 tests passed
9 tests failed
17 tests skipped
0 tests ignored
0 tests passed unexpectedly
6 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
For a full summary look at the liquibase artifacts
An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89368 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #87586 roachtest: liquibase failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6848253?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6848253?buildTab=artifacts#/liquibase) on master @ [e06d2286b011096526eda7f2d7f7bb7acea0ae84](https://github.com/cockroachdb/cockroach/commits/e06d2286b011096526eda7f2d7f7bb7acea0ae84):


```
test artifacts and logs in: /artifacts/liquibase/run_1
(orm_helpers.go:191).summarizeFailed: 
Tests run on Cockroach v22.2.0-alpha.3-1023-ge06d2286b0
Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
47 Total Tests Run
38 tests passed
9 tests failed
17 tests skipped
0 tests ignored
0 tests passed unexpectedly
6 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
For a full summary look at the liquibase artifacts
An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89368 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #87586 roachtest: liquibase failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6859829?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6859829?buildTab=artifacts#/liquibase) on master @ [6e06a05eba118f512087da9aec3147b3446c2faf](https://github.com/cockroachdb/cockroach/commits/6e06a05eba118f512087da9aec3147b3446c2faf):


```
test artifacts and logs in: /artifacts/liquibase/run_1
(orm_helpers.go:191).summarizeFailed: 
Tests run on Cockroach v22.2.0-alpha.3-1027-g6e06a05eba
Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
47 Total Tests Run
38 tests passed
9 tests failed
17 tests skipped
0 tests ignored
0 tests passed unexpectedly
6 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
For a full summary look at the liquibase artifacts
An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89368 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #87586 roachtest: liquibase failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6879260?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6879260?buildTab=artifacts#/liquibase) on master @ [8a8855eaf1a34223523d7631600481f1c2577a96](https://github.com/cockroachdb/cockroach/commits/8a8855eaf1a34223523d7631600481f1c2577a96):


```
test artifacts and logs in: /artifacts/liquibase/run_1
(orm_helpers.go:191).summarizeFailed: 
Tests run on Cockroach v22.2.0-alpha.3-1066-g8a8855eaf1
Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
47 Total Tests Run
38 tests passed
9 tests failed
17 tests skipped
0 tests ignored
0 tests passed unexpectedly
6 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
For a full summary look at the liquibase artifacts
An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89368 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #87586 roachtest: liquibase failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6897503?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6897503?buildTab=artifacts#/liquibase) on master @ [d1962deb662c115ceec158a0c09d3f797eca3e71](https://github.com/cockroachdb/cockroach/commits/d1962deb662c115ceec158a0c09d3f797eca3e71):


```
test artifacts and logs in: /artifacts/liquibase/run_1
(orm_helpers.go:191).summarizeFailed: 
Tests run on Cockroach v22.2.0-alpha.3-1102-gd1962deb66
Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
47 Total Tests Run
38 tests passed
9 tests failed
17 tests skipped
0 tests ignored
0 tests passed unexpectedly
6 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
For a full summary look at the liquibase artifacts
An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89368 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #87586 roachtest: liquibase failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6918230?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6918230?buildTab=artifacts#/liquibase) on master @ [4c2c7da2abef4875c44ade99fd4781c85be69637](https://github.com/cockroachdb/cockroach/commits/4c2c7da2abef4875c44ade99fd4781c85be69637):


```
test artifacts and logs in: /artifacts/liquibase/run_1
(orm_helpers.go:191).summarizeFailed: 
Tests run on Cockroach v22.2.0-alpha.3-1134-g4c2c7da2ab
Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
47 Total Tests Run
38 tests passed
9 tests failed
17 tests skipped
0 tests ignored
0 tests passed unexpectedly
6 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
For a full summary look at the liquibase artifacts
An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89368 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #87586 roachtest: liquibase failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6937671?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6937671?buildTab=artifacts#/liquibase) on master @ [d6c28f56d38c747dbd6fc64359e6255c873083c5](https://github.com/cockroachdb/cockroach/commits/d6c28f56d38c747dbd6fc64359e6255c873083c5):


```
test artifacts and logs in: /artifacts/liquibase/run_1
(orm_helpers.go:191).summarizeFailed: 
Tests run on Cockroach v22.2.0-alpha.3-1172-gd6c28f56d3
Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
47 Total Tests Run
38 tests passed
9 tests failed
17 tests skipped
0 tests ignored
0 tests passed unexpectedly
6 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
For a full summary look at the liquibase artifacts
An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89368 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #87586 roachtest: liquibase failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6949067?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6949067?buildTab=artifacts#/liquibase) on master @ [d6c28f56d38c747dbd6fc64359e6255c873083c5](https://github.com/cockroachdb/cockroach/commits/d6c28f56d38c747dbd6fc64359e6255c873083c5):


```
test artifacts and logs in: /artifacts/liquibase/run_1
(orm_helpers.go:191).summarizeFailed: 
Tests run on Cockroach v22.2.0-alpha.3-1172-gd6c28f56d3
Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
47 Total Tests Run
38 tests passed
9 tests failed
17 tests skipped
0 tests ignored
0 tests passed unexpectedly
6 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
For a full summary look at the liquibase artifacts
An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89368 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #87586 roachtest: liquibase failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6953743?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6953743?buildTab=artifacts#/liquibase) on master @ [7be0b20edbc336200c1510a9c6f1d76ae2f92c3a](https://github.com/cockroachdb/cockroach/commits/7be0b20edbc336200c1510a9c6f1d76ae2f92c3a):


```
test artifacts and logs in: /artifacts/liquibase/run_1
(orm_helpers.go:191).summarizeFailed: 
Tests run on Cockroach v22.2.0-alpha.3-1196-g7be0b20edb
Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
47 Total Tests Run
38 tests passed
9 tests failed
17 tests skipped
0 tests ignored
0 tests passed unexpectedly
6 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
For a full summary look at the liquibase artifacts
An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89368 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #87586 roachtest: liquibase failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6964755?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6964755?buildTab=artifacts#/liquibase) on master @ [7be0b20edbc336200c1510a9c6f1d76ae2f92c3a](https://github.com/cockroachdb/cockroach/commits/7be0b20edbc336200c1510a9c6f1d76ae2f92c3a):


```
test artifacts and logs in: /artifacts/liquibase/run_1
(orm_helpers.go:191).summarizeFailed: 
Tests run on Cockroach v22.2.0-alpha.3-1196-g7be0b20edb
Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
47 Total Tests Run
38 tests passed
9 tests failed
17 tests skipped
0 tests ignored
0 tests passed unexpectedly
6 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
For a full summary look at the liquibase artifacts
An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89368 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #87586 roachtest: liquibase failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6975891?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6975891?buildTab=artifacts#/liquibase) on master @ [6b68e55309343a26300296bf42237a70d47297b2](https://github.com/cockroachdb/cockroach/commits/6b68e55309343a26300296bf42237a70d47297b2):


```
test artifacts and logs in: /artifacts/liquibase/run_1
(orm_helpers.go:191).summarizeFailed: 
Tests run on Cockroach v22.2.0-alpha.3-1200-g6b68e55309
Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
47 Total Tests Run
38 tests passed
9 tests failed
17 tests skipped
0 tests ignored
0 tests passed unexpectedly
6 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
For a full summary look at the liquibase artifacts
An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89368 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #87586 roachtest: liquibase failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6994309?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6994309?buildTab=artifacts#/liquibase) on master @ [762c1b86fe1c3a70338cee4a91c9e2e4c5e0fcfe](https://github.com/cockroachdb/cockroach/commits/762c1b86fe1c3a70338cee4a91c9e2e4c5e0fcfe):


```
test artifacts and logs in: /artifacts/liquibase/run_1
(orm_helpers.go:191).summarizeFailed: 
Tests run on Cockroach v22.2.0-alpha.3-1244-g762c1b86fe
Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
47 Total Tests Run
38 tests passed
9 tests failed
17 tests skipped
0 tests ignored
0 tests passed unexpectedly
6 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
For a full summary look at the liquibase artifacts
An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89368 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #87586 roachtest: liquibase failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7014808?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7014808?buildTab=artifacts#/liquibase) on master @ [bb4f8cd5396e73c31d370c0d0ac1881263c5156f](https://github.com/cockroachdb/cockroach/commits/bb4f8cd5396e73c31d370c0d0ac1881263c5156f):


```
test artifacts and logs in: /artifacts/liquibase/run_1
(orm_helpers.go:191).summarizeFailed: 
Tests run on Cockroach v22.2.0-alpha.3-1288-gbb4f8cd539
Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
47 Total Tests Run
38 tests passed
9 tests failed
17 tests skipped
0 tests ignored
0 tests passed unexpectedly
6 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
For a full summary look at the liquibase artifacts
An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89368 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #87586 roachtest: liquibase failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7033383?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7033383?buildTab=artifacts#/liquibase) on master @ [16b020292dbcdf9699da531764929ebdc8e48c43](https://github.com/cockroachdb/cockroach/commits/16b020292dbcdf9699da531764929ebdc8e48c43):


```
test artifacts and logs in: /artifacts/liquibase/run_1
(orm_helpers.go:191).summarizeFailed: 
Tests run on Cockroach v22.2.0-alpha.3-1326-g16b020292d
Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
47 Total Tests Run
38 tests passed
9 tests failed
17 tests skipped
0 tests ignored
0 tests passed unexpectedly
6 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
For a full summary look at the liquibase artifacts
An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89368 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #87586 roachtest: liquibase failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7053206?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7053206?buildTab=artifacts#/liquibase) on master @ [0a11b74e2343e5201900fd2643724774fcf3fcd1](https://github.com/cockroachdb/cockroach/commits/0a11b74e2343e5201900fd2643724774fcf3fcd1):


```
test artifacts and logs in: /artifacts/liquibase/run_1
(orm_helpers.go:191).summarizeFailed: 
Tests run on Cockroach v22.2.0-alpha.3-1366-g0a11b74e23
Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
47 Total Tests Run
38 tests passed
9 tests failed
17 tests skipped
0 tests ignored
0 tests passed unexpectedly
6 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
For a full summary look at the liquibase artifacts
An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89368 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #87586 roachtest: liquibase failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7070913?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7070913?buildTab=artifacts#/liquibase) on master @ [a3b578466bcdcc5d2bbea628d33c337d87b9f4b8](https://github.com/cockroachdb/cockroach/commits/a3b578466bcdcc5d2bbea628d33c337d87b9f4b8):


```
test artifacts and logs in: /artifacts/liquibase/run_1
(orm_helpers.go:191).summarizeFailed: 
Tests run on Cockroach v22.2.0-alpha.3-1397-ga3b578466b
Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
47 Total Tests Run
38 tests passed
9 tests failed
17 tests skipped
0 tests ignored
0 tests passed unexpectedly
6 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
For a full summary look at the liquibase artifacts
An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89368 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #87586 roachtest: liquibase failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7082415?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7082415?buildTab=artifacts#/liquibase) on master @ [cdf4b18048a9e29019125c64bea75c8d50fa3774](https://github.com/cockroachdb/cockroach/commits/cdf4b18048a9e29019125c64bea75c8d50fa3774):


```
test artifacts and logs in: /artifacts/liquibase/run_1
(orm_helpers.go:191).summarizeFailed: 
Tests run on Cockroach v22.2.0-alpha.3-1401-gcdf4b18048
Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
47 Total Tests Run
38 tests passed
9 tests failed
17 tests skipped
0 tests ignored
0 tests passed unexpectedly
6 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
For a full summary look at the liquibase artifacts
An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89368 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #87586 roachtest: liquibase failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7114342?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7114342?buildTab=artifacts#/liquibase) on master @ [c5decaeb69155399647909163451f3d8ca7858b6](https://github.com/cockroachdb/cockroach/commits/c5decaeb69155399647909163451f3d8ca7858b6):


```
test artifacts and logs in: /artifacts/liquibase/run_1
(orm_helpers.go:191).summarizeFailed: 
Tests run on Cockroach v22.2.0-alpha.3-1444-gc5decaeb69
Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
47 Total Tests Run
38 tests passed
9 tests failed
17 tests skipped
0 tests ignored
0 tests passed unexpectedly
6 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
For a full summary look at the liquibase artifacts
An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89368 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #87586 roachtest: liquibase failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7136814?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7136814?buildTab=artifacts#/liquibase) on master @ [1b1c8da55be48c174b7b370b305f42622546209f](https://github.com/cockroachdb/cockroach/commits/1b1c8da55be48c174b7b370b305f42622546209f):


```
test artifacts and logs in: /artifacts/liquibase/run_1
(test_impl.go:297).Fatalf: 
Tests run on Cockroach v22.2.0-alpha.3-1471-g1b1c8da55b
Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
47 Total Tests Run
38 tests passed
9 tests failed
17 tests skipped
0 tests ignored
0 tests passed unexpectedly
6 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
For a full summary look at the liquibase artifacts
An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89368 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #87586 roachtest: liquibase failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7161019?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7161019?buildTab=artifacts#/liquibase) on master @ [100452314211aedd1e295bba6979df71d9052629](https://github.com/cockroachdb/cockroach/commits/100452314211aedd1e295bba6979df71d9052629):


```
test artifacts and logs in: /artifacts/liquibase/run_1
(test_impl.go:297).Fatalf: 
Tests run on Cockroach v22.2.0-alpha.3-1549-g1004523142
Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
47 Total Tests Run
38 tests passed
9 tests failed
17 tests skipped
0 tests ignored
0 tests passed unexpectedly
6 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
For a full summary look at the liquibase artifacts
An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89368 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #87586 roachtest: liquibase failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7182921?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7182921?buildTab=artifacts#/liquibase) on master @ [d39558f51ded0411e90ef1d8aa05106930b36667](https://github.com/cockroachdb/cockroach/commits/d39558f51ded0411e90ef1d8aa05106930b36667):


```
test artifacts and logs in: /artifacts/liquibase/run_1
(test_impl.go:297).Fatalf: 
Tests run on Cockroach v22.2.0-alpha.3-1592-gd39558f51d
Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
47 Total Tests Run
38 tests passed
9 tests failed
17 tests skipped
0 tests ignored
0 tests passed unexpectedly
6 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
For a full summary look at the liquibase artifacts
An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89368 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #87586 roachtest: liquibase failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7201916?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7201916?buildTab=artifacts#/liquibase) on master @ [0c1c3e7777b28a30ebe41428fb173f0156e8968c](https://github.com/cockroachdb/cockroach/commits/0c1c3e7777b28a30ebe41428fb173f0156e8968c):


```
test artifacts and logs in: /artifacts/liquibase/run_1
(test_impl.go:297).Fatalf: 
Tests run on Cockroach v22.2.0-alpha.3-1630-g0c1c3e7777
Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
47 Total Tests Run
38 tests passed
9 tests failed
17 tests skipped
0 tests ignored
0 tests passed unexpectedly
6 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
For a full summary look at the liquibase artifacts
An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89368 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #87586 roachtest: liquibase failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7213327?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7213327?buildTab=artifacts#/liquibase) on master @ [0c1c3e7777b28a30ebe41428fb173f0156e8968c](https://github.com/cockroachdb/cockroach/commits/0c1c3e7777b28a30ebe41428fb173f0156e8968c):


```
test artifacts and logs in: /artifacts/liquibase/run_1
(test_impl.go:297).Fatalf: 
Tests run on Cockroach v22.2.0-alpha.3-1630-g0c1c3e7777
Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
47 Total Tests Run
38 tests passed
9 tests failed
17 tests skipped
0 tests ignored
0 tests passed unexpectedly
6 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
For a full summary look at the liquibase artifacts
An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89368 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #87586 roachtest: liquibase failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7225361?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7225361?buildTab=artifacts#/liquibase) on master @ [2d926e68000df659f282d4e4477329867b9a3323](https://github.com/cockroachdb/cockroach/commits/2d926e68000df659f282d4e4477329867b9a3323):


```
test artifacts and logs in: /artifacts/liquibase/run_1
(test_impl.go:297).Fatalf: 
Tests run on Cockroach v22.2.0-alpha.3-1636-g2d926e6800
Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
47 Total Tests Run
38 tests passed
9 tests failed
17 tests skipped
0 tests ignored
0 tests passed unexpectedly
6 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
For a full summary look at the liquibase artifacts
An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89368 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #87586 roachtest: liquibase failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7247923?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7247923?buildTab=artifacts#/liquibase) on master @ [18675b7268a89dc646f37b7208c2eaa319f7741c](https://github.com/cockroachdb/cockroach/commits/18675b7268a89dc646f37b7208c2eaa319f7741c):


```
test artifacts and logs in: /artifacts/liquibase/run_1
(test_impl.go:297).Fatalf: 
Tests run on Cockroach v22.2.0-alpha.3-1703-g18675b7268
Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
47 Total Tests Run
38 tests passed
9 tests failed
17 tests skipped
0 tests ignored
0 tests passed unexpectedly
6 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
For a full summary look at the liquibase artifacts
An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89368 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #87586 roachtest: liquibase failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7269456?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7269456?buildTab=artifacts#/liquibase) on master @ [bec24eb9cfe2cc48c45772f1e88d387fe37944c9](https://github.com/cockroachdb/cockroach/commits/bec24eb9cfe2cc48c45772f1e88d387fe37944c9):


```
test artifacts and logs in: /artifacts/liquibase/run_1
(test_impl.go:297).Fatalf: 
Tests run on Cockroach v22.2.0-alpha.3-1749-gbec24eb9cf
Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
47 Total Tests Run
38 tests passed
9 tests failed
17 tests skipped
0 tests ignored
0 tests passed unexpectedly
6 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
For a full summary look at the liquibase artifacts
An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89368 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #87586 roachtest: liquibase failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7288517?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7288517?buildTab=artifacts#/liquibase) on master @ [9c9d55d707ad9a768027e9b7a3775c7c7cde8de7](https://github.com/cockroachdb/cockroach/commits/9c9d55d707ad9a768027e9b7a3775c7c7cde8de7):


```
test artifacts and logs in: /artifacts/liquibase/run_1
(test_impl.go:297).Fatalf: 
Tests run on Cockroach v22.2.0-alpha.3-1768-g9c9d55d707
Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
47 Total Tests Run
38 tests passed
9 tests failed
17 tests skipped
0 tests ignored
0 tests passed unexpectedly
6 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
For a full summary look at the liquibase artifacts
An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89368 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #87586 roachtest: liquibase failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7309179?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7309179?buildTab=artifacts#/liquibase) on master @ [01cfd27375aaf4633b6f6722924c8e547af4959d](https://github.com/cockroachdb/cockroach/commits/01cfd27375aaf4633b6f6722924c8e547af4959d):


```
test artifacts and logs in: /artifacts/liquibase/run_1
(test_impl.go:297).Fatalf: 
Tests run on Cockroach v22.2.0-alpha.3-1813-g01cfd27375
Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
47 Total Tests Run
38 tests passed
9 tests failed
17 tests skipped
0 tests ignored
0 tests passed unexpectedly
6 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
For a full summary look at the liquibase artifacts
An updated blocklist (liquibaseBlocklist22_2) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89368 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #87586 roachtest: liquibase failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7330110?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7330110?buildTab=artifacts#/liquibase) on master @ [2c0129ce2507e708382cda830d1e47817a299c2e](https://github.com/cockroachdb/cockroach/commits/2c0129ce2507e708382cda830d1e47817a299c2e):


```
test artifacts and logs in: /artifacts/liquibase/run_1
(test_impl.go:297).Fatalf: 
Tests run on Cockroach v23.1.0-alpha.00000000-35-g2c0129ce25
Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
47 Total Tests Run
38 tests passed
9 tests failed
17 tests skipped
0 tests ignored
0 tests passed unexpectedly
6 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
For a full summary look at the liquibase artifacts
An updated blocklist (liquibaseBlocklist23_1) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89368 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #87586 roachtest: liquibase failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7342303?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7342303?buildTab=artifacts#/liquibase) on master @ [c9094fa1a1ef8d44a2d5299d1d7380b34a06241b](https://github.com/cockroachdb/cockroach/commits/c9094fa1a1ef8d44a2d5299d1d7380b34a06241b):


```
test artifacts and logs in: /artifacts/liquibase/run_1
(test_impl.go:297).Fatalf: 
Tests run on Cockroach v23.1.0-alpha.00000000-46-gc9094fa1a1
Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
47 Total Tests Run
38 tests passed
9 tests failed
17 tests skipped
0 tests ignored
0 tests passed unexpectedly
6 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
For a full summary look at the liquibase artifacts
An updated blocklist (liquibaseBlocklist23_1) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89368 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #87586 roachtest: liquibase failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7354413?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7354413?buildTab=artifacts#/liquibase) on master @ [4c828ca5df8ea70b92ba58c6cb4868303cee1262](https://github.com/cockroachdb/cockroach/commits/4c828ca5df8ea70b92ba58c6cb4868303cee1262):


```
test artifacts and logs in: /artifacts/liquibase/run_1
(test_impl.go:297).Fatalf: 
Tests run on Cockroach v23.1.0-alpha.00000000-48-g4c828ca5df
Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
47 Total Tests Run
38 tests passed
9 tests failed
17 tests skipped
0 tests ignored
0 tests passed unexpectedly
6 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
For a full summary look at the liquibase artifacts
An updated blocklist (liquibaseBlocklist23_1) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89368 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #87586 roachtest: liquibase failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7377852?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7377852?buildTab=artifacts#/liquibase) on master @ [ff8f7eda167887e1f11bbcd9029aa4cc2d28085e](https://github.com/cockroachdb/cockroach/commits/ff8f7eda167887e1f11bbcd9029aa4cc2d28085e):


```
test artifacts and logs in: /artifacts/liquibase/run_1
(test_impl.go:297).Fatalf: 
Tests run on Cockroach v23.1.0-alpha.00000000-69-gff8f7eda16
Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
47 Total Tests Run
38 tests passed
9 tests failed
17 tests skipped
0 tests ignored
0 tests passed unexpectedly
6 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
For a full summary look at the liquibase artifacts
An updated blocklist (liquibaseBlocklist23_1) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89368 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #87586 roachtest: liquibase failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7402563?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7402563?buildTab=artifacts#/liquibase) on master @ [ee4fdb7fff683a7916ee585ac0c1e22d5d9a4412](https://github.com/cockroachdb/cockroach/commits/ee4fdb7fff683a7916ee585ac0c1e22d5d9a4412):


```
test artifacts and logs in: /artifacts/liquibase/run_1
(test_impl.go:297).Fatalf: 
Tests run on Cockroach v23.1.0-alpha.00000000-144-gee4fdb7fff
Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
47 Total Tests Run
38 tests passed
9 tests failed
17 tests skipped
0 tests ignored
0 tests passed unexpectedly
6 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
For a full summary look at the liquibase artifacts
An updated blocklist (liquibaseBlocklist23_1) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89368 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #87586 roachtest: liquibase failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7448348?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7448348?buildTab=artifacts#/liquibase) on master @ [d136c1fa50fa507c2232d7b215d98b88a83649c5](https://github.com/cockroachdb/cockroach/commits/d136c1fa50fa507c2232d7b215d98b88a83649c5):


```
test artifacts and logs in: /artifacts/liquibase/run_1
(test_impl.go:297).Fatalf: 
Tests run on Cockroach v23.1.0-alpha.00000000-223-gd136c1fa50
Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
47 Total Tests Run
38 tests passed
9 tests failed
17 tests skipped
0 tests ignored
0 tests passed unexpectedly
6 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
For a full summary look at the liquibase artifacts
An updated blocklist (liquibaseBlocklist23_1) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89368 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #87586 roachtest: liquibase failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7466963?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7466963?buildTab=artifacts#/liquibase) on master @ [fb4014a31b9b8d8235dc48de52196e64b185f490](https://github.com/cockroachdb/cockroach/commits/fb4014a31b9b8d8235dc48de52196e64b185f490):


```
test artifacts and logs in: /artifacts/liquibase/run_1
(test_impl.go:297).Fatalf: 
Tests run on Cockroach v23.1.0-alpha.00000000-258-gfb4014a31b
Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
47 Total Tests Run
38 tests passed
9 tests failed
17 tests skipped
0 tests ignored
0 tests passed unexpectedly
6 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
For a full summary look at the liquibase artifacts
An updated blocklist (liquibaseBlocklist23_1) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89368 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #87586 roachtest: liquibase failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7478515?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7478515?buildTab=artifacts#/liquibase) on master @ [af18b5d2f8ec09d6f3c3092be9ef5fe3f460724d](https://github.com/cockroachdb/cockroach/commits/af18b5d2f8ec09d6f3c3092be9ef5fe3f460724d):


```
test artifacts and logs in: /artifacts/liquibase/run_1
(test_impl.go:297).Fatalf: 
Tests run on Cockroach v23.1.0-alpha.00000000-265-gaf18b5d2f8
Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
47 Total Tests Run
38 tests passed
9 tests failed
17 tests skipped
0 tests ignored
0 tests passed unexpectedly
6 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
For a full summary look at the liquibase artifacts
An updated blocklist (liquibaseBlocklist23_1) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89368 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #87586 roachtest: liquibase failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7490747?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7490747?buildTab=artifacts#/liquibase) on master @ [7f77f57a768a1bc488629234126e9e456bb6ddbb](https://github.com/cockroachdb/cockroach/commits/7f77f57a768a1bc488629234126e9e456bb6ddbb):


```
test artifacts and logs in: /artifacts/liquibase/run_1
(test_impl.go:297).Fatalf: 
Tests run on Cockroach v23.1.0-alpha.00000000-269-g7f77f57a76
Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
47 Total Tests Run
38 tests passed
9 tests failed
17 tests skipped
0 tests ignored
0 tests passed unexpectedly
6 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
For a full summary look at the liquibase artifacts
An updated blocklist (liquibaseBlocklist23_1) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89368 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #87586 roachtest: liquibase failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7512756?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7512756?buildTab=artifacts#/liquibase) on master @ [47c7b3a1bc047fc3e481cf12166885b39519c022](https://github.com/cockroachdb/cockroach/commits/47c7b3a1bc047fc3e481cf12166885b39519c022):


```
test artifacts and logs in: /artifacts/liquibase/run_1
(test_impl.go:297).Fatalf: 
Tests run on Cockroach v23.1.0-alpha.00000000-310-g47c7b3a1bc
Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
47 Total Tests Run
38 tests passed
9 tests failed
17 tests skipped
0 tests ignored
0 tests passed unexpectedly
6 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
For a full summary look at the liquibase artifacts
An updated blocklist (liquibaseBlocklist23_1) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89368 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #87586 roachtest: liquibase failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7534902?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7534902?buildTab=artifacts#/liquibase) on master @ [d98d195dfa7a3480ce3e07657aa870692fe71cea](https://github.com/cockroachdb/cockroach/commits/d98d195dfa7a3480ce3e07657aa870692fe71cea):


```
test artifacts and logs in: /artifacts/liquibase/run_1
(test_impl.go:297).Fatalf: 
Tests run on Cockroach v23.1.0-alpha.00000000-356-gd98d195dfa
Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
47 Total Tests Run
38 tests passed
9 tests failed
17 tests skipped
0 tests ignored
0 tests passed unexpectedly
6 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
For a full summary look at the liquibase artifacts
An updated blocklist (liquibaseBlocklist23_1) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89368 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #87586 roachtest: liquibase failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7555547?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7555547?buildTab=artifacts#/liquibase) on master @ [ea7c52e5a930be203318db6e338edd5dde7537bd](https://github.com/cockroachdb/cockroach/commits/ea7c52e5a930be203318db6e338edd5dde7537bd):


```
test artifacts and logs in: /artifacts/liquibase/run_1
(test_impl.go:297).Fatalf: 
Tests run on Cockroach v23.1.0-alpha.00000000-425-gea7c52e5a9
Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
47 Total Tests Run
38 tests passed
9 tests failed
17 tests skipped
0 tests ignored
0 tests passed unexpectedly
6 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
For a full summary look at the liquibase artifacts
An updated blocklist (liquibaseBlocklist23_1) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89368 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #87586 roachtest: liquibase failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7574283?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7574283?buildTab=artifacts#/liquibase) on master @ [a035d2e6e7ea57b30115ecc8ee1a5d553a9e3412](https://github.com/cockroachdb/cockroach/commits/a035d2e6e7ea57b30115ecc8ee1a5d553a9e3412):


```
test artifacts and logs in: /artifacts/liquibase/run_1
(test_impl.go:297).Fatalf: 
Tests run on Cockroach v23.1.0-alpha.00000000-474-ga035d2e6e7
Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
47 Total Tests Run
38 tests passed
9 tests failed
17 tests skipped
0 tests ignored
0 tests passed unexpectedly
6 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
For a full summary look at the liquibase artifacts
An updated blocklist (liquibaseBlocklist23_1) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89368 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #87586 roachtest: liquibase failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7593502?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7593502?buildTab=artifacts#/liquibase) on master @ [8357abb668a5adaff781343b394b162fb1b66c6e](https://github.com/cockroachdb/cockroach/commits/8357abb668a5adaff781343b394b162fb1b66c6e):


```
test artifacts and logs in: /artifacts/liquibase/run_1
(test_impl.go:297).Fatalf: 
Tests run on Cockroach v23.1.0-alpha.00000000-506-g8357abb668
Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
47 Total Tests Run
38 tests passed
9 tests failed
17 tests skipped
0 tests ignored
0 tests passed unexpectedly
6 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
For a full summary look at the liquibase artifacts
An updated blocklist (liquibaseBlocklist23_1) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89368 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #87586 roachtest: liquibase failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7605127?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7605127?buildTab=artifacts#/liquibase) on master @ [cfb5ae9a96e1770daa4aef1615a46e212b561a84](https://github.com/cockroachdb/cockroach/commits/cfb5ae9a96e1770daa4aef1615a46e212b561a84):


```
test artifacts and logs in: /artifacts/liquibase/run_1
(test_impl.go:297).Fatalf: 
Tests run on Cockroach v23.1.0-alpha.00000000-516-gcfb5ae9a96
Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
47 Total Tests Run
38 tests passed
9 tests failed
17 tests skipped
0 tests ignored
0 tests passed unexpectedly
6 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
For a full summary look at the liquibase artifacts
An updated blocklist (liquibaseBlocklist23_1) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89368 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #87586 roachtest: liquibase failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7616997?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7616997?buildTab=artifacts#/liquibase) on master @ [f694132d6ed73c707576a28f6e784fda8ea13c63](https://github.com/cockroachdb/cockroach/commits/f694132d6ed73c707576a28f6e784fda8ea13c63):


```
test artifacts and logs in: /artifacts/liquibase/run_1
(test_impl.go:297).Fatalf: 
Tests run on Cockroach v23.1.0-alpha.00000000-522-gf694132d6e
Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
47 Total Tests Run
38 tests passed
9 tests failed
17 tests skipped
0 tests ignored
0 tests passed unexpectedly
6 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
For a full summary look at the liquibase artifacts
An updated blocklist (liquibaseBlocklist23_1) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89368 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #87586 roachtest: liquibase failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #83313 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7697774?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7697774?buildTab=artifacts#/liquibase) on master @ [1a6e9f885baa124d5ff2996adb966ea15a1a9b2b](https://github.com/cockroachdb/cockroach/commits/1a6e9f885baa124d5ff2996adb966ea15a1a9b2b):


```
test artifacts and logs in: /artifacts/liquibase/run_1
(test_impl.go:297).Fatalf: 
Tests run on Cockroach v23.1.0-alpha.00000000-648-g1a6e9f885b
Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
47 Total Tests Run
38 tests passed
9 tests failed
17 tests skipped
0 tests ignored
0 tests passed unexpectedly
6 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
For a full summary look at the liquibase artifacts
An updated blocklist (liquibaseBlocklist23_1) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89368 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #87586 roachtest: liquibase failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7709089?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7709089?buildTab=artifacts#/liquibase) on master @ [1a6e9f885baa124d5ff2996adb966ea15a1a9b2b](https://github.com/cockroachdb/cockroach/commits/1a6e9f885baa124d5ff2996adb966ea15a1a9b2b):


```
test artifacts and logs in: /artifacts/liquibase/run_1
(test_impl.go:297).Fatalf: 
Tests run on Cockroach v23.1.0-alpha.00000000-648-g1a6e9f885b
Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
47 Total Tests Run
38 tests passed
9 tests failed
17 tests skipped
0 tests ignored
0 tests passed unexpectedly
6 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
For a full summary look at the liquibase artifacts
An updated blocklist (liquibaseBlocklist23_1) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89368 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #87586 roachtest: liquibase failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7720724?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7720724?buildTab=artifacts#/liquibase) on master @ [1a6e9f885baa124d5ff2996adb966ea15a1a9b2b](https://github.com/cockroachdb/cockroach/commits/1a6e9f885baa124d5ff2996adb966ea15a1a9b2b):


```
test artifacts and logs in: /artifacts/liquibase/run_1
(test_impl.go:297).Fatalf: 
Tests run on Cockroach v23.1.0-alpha.00000000-648-g1a6e9f885b
Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
47 Total Tests Run
38 tests passed
9 tests failed
17 tests skipped
0 tests ignored
0 tests passed unexpectedly
6 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
For a full summary look at the liquibase artifacts
An updated blocklist (liquibaseBlocklist23_1) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89368 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #87586 roachtest: liquibase failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7742763?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7742763?buildTab=artifacts#/liquibase) on master @ [2b595c5214aead3e7d4f47f9a1dedce57dba6e7f](https://github.com/cockroachdb/cockroach/commits/2b595c5214aead3e7d4f47f9a1dedce57dba6e7f):


```
test artifacts and logs in: /artifacts/liquibase/run_1
(test_impl.go:297).Fatalf: 
Tests run on Cockroach v23.1.0-alpha.00000000-673-g2b595c5214
Tests run against liquibase 1790ddef2d0339c5c96839ac60ac424c130dadd8
47 Total Tests Run
38 tests passed
9 tests failed
17 tests skipped
0 tests ignored
0 tests passed unexpectedly
6 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2 - unknown (unexpected)
For a full summary look at the liquibase artifacts
An updated blocklist (liquibaseBlocklist23_1) is available in the artifacts' liquibase log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89368 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #87586 roachtest: liquibase failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #83048 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

--- a/pkg/cmd/roachtest/tests/liquibase.go
+++ b/pkg/cmd/roachtest/tests/liquibase.go
@@ -101,10 +101,9 @@ func registerLiquibase(r registry.Registry) {
 		}
 
 		t.Status("running liquibase test harness")
-		blocklistName, expectedFailures, _, ignoreList := liquibaseBlocklists.getLists(version)
-		if expectedFailures == nil {
-			t.Fatalf("No hibernate blocklist defined for cockroach version %s", version)
-		}
+		const blocklistName = "liquibaseBlocklist"
+		expectedFailures := liquibaseBlocklist
+		ignoreList := liquibaseIgnorelist
 
 		const (
 			repoDir     = "/mnt/data1/liquibase-test-harness"

--- a/pkg/cmd/roachtest/tests/liquibase_blocklist.go
+++ b/pkg/cmd/roachtest/tests/liquibase_blocklist.go
@@ -10,41 +10,16 @@
 
 package tests
 
-var liquibaseBlocklists = blocklistsForVersion{
-	{"v20.2", "liquibaseBlocklist20_2", liquibaseBlocklist20_2, "liquibaseIgnorelist20_2", liquibaseIgnorelist20_2},
-	{"v21.1", "liquibaseBlocklist21_1", liquibaseBlocklist21_1, "liquibaseIgnorelist21_1", liquibaseIgnorelist21_1},
-	{"v21.2", "liquibaseBlocklist21_2", liquibaseBlocklist21_2, "liquibaseIgnorelist21_2", liquibaseIgnorelist21_2},
-	{"v22.1", "liquibaseBlocklist22_1", liquibaseBlocklist22_1, "liquibaseIgnorelist21_2", liquibaseIgnorelist22_1},
-	{"v22.2", "liquibaseBlocklist22_2", liquibaseBlocklist22_2, "liquibaseIgnorelist21_2", liquibaseIgnorelist22_2},
-	{"v23.1", "liquibaseBlocklist23_1", liquibaseBlocklist23_1, "liquibaseIgnorelist23_1", liquibaseIgnorelist23_1},
+var liquibaseBlocklist = blocklist{
+	"liquibase.harness.change.ChangeObjectTests.apply addAutoIncrement against cockroachdb 20.2":            "unknown",
+	"liquibase.harness.change.ChangeObjectTests.apply addCheckConstraint against cockroachdb 20.2":          "unknown",
+	"liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2": "unknown",
+	"liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 20.2":               "unknown",
+	"liquibase.harness.change.ChangeObjectTests.apply createPackage against cockroachdb 20.2":               "unknown",
+	"liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 20.2":              "unknown",
+	"liquibase.harness.change.ChangeObjectTests.apply dropCheckConstraint against cockroachdb 20.2":         "unknown",
+	"liquibase.harness.change.ChangeObjectTests.apply dropSequence against cockroachdb 20.2":                "unknown",
+	"liquibase.harness.change.ChangeObjectTests.apply renameSequence against cockroachdb 20.2":              "unknown",
 }
 
-var liquibaseBlocklist23_1 = liquibaseBlocklist22_2
-
-var liquibaseBlocklist22_2 = liquibaseBlocklist22_1
-
-var liquibaseBlocklist22_1 = blocklist{
-	"liquibase.harness.change.ChangeObjectTests.apply addCheckConstraint against cockroachdb 20.2":  "unknown",
-	"liquibase.harness.change.ChangeObjectTests.apply createPackage against cockroachdb 20.2":       "unknown",
-	"liquibase.harness.change.ChangeObjectTests.apply dropCheckConstraint against cockroachdb 20.2": "unknown",
-}
-
-var liquibaseBlocklist21_2 = blocklist{
-	"liquibase.harness.change.ChangeObjectTests.apply addDefaultValueSequenceNext against cockroachdb 20.2; verify generated SQL and DB snapshot": "",
-}
-
-var liquibaseBlocklist21_1 = liquibaseBlocklist20_2
-
-var liquibaseBlocklist20_2 = blocklist{}
-
-var liquibaseIgnorelist23_1 = liquibaseIgnorelist22_2
-
-var liquibaseIgnorelist22_2 = liquibaseIgnorelist22_1
-
-var liquibaseIgnorelist22_1 = liquibaseIgnorelist21_2
-
-var liquibaseIgnorelist21_2 = liquibaseIgnorelist21_1
-
-var liquibaseIgnorelist21_1 = liquibaseIgnorelist20_2
-
-var liquibaseIgnorelist20_2 = blocklist{}
+var liquibaseIgnorelist = blocklist{}

